// crawlingModule/lib/dataHelper.js
(function () {

    var output = {};


    output.modeltonModelName = {
        'R140W-7' : 'Excavator(12~15.5t)',
        'R140W-7A' : 'Excavator(12~15.5t)',
        'R140W-9' : 'Excavator(12~15.5t)',
        'R170W-7' : 'Excavator(16~19t)',
        'R170W-7A' : 'Excavator(16~19t)',
        'R170W-9' : 'Excavator(16~19t)',
        'R200W-7' : 'Excavator(20~27t)',
        'R200W-7A' : 'Excavator(20~27t)',
        'R210W-9' : 'Excavator(20~27t)',
        'R55W-7' : 'Excavator(4.5~6t)',
        'R55W-7A' : 'Excavator(4.5~6t)',
        'R55W-9' : 'Excavator(4.5~6t)',
        'R95W-3' : 'Excavator(7~11t)',
        'R140W-7' : 'Excavator(12~15.5t)',
        'R140W-7A' : 'Excavator(12~15.5t)',
        'R170W-7' : 'Excavator(16~19t)',
        'R170W-7A' : 'Excavator(16~19t)',
        'R200W-7' : 'Excavator(20~27t)',
        'R200W-7A' : 'Excavator(20~27t)',
        'R55W-7' : 'Excavator(4.5~6t)',
        'R55W-7A' : 'Excavator(4.5~6t)',
        'R140W-9' : 'Excavator(12~15.5t)',
        'R170W-9' : 'Excavator(16~19t)',
        'R210W-9' : 'Excavator(20~27t)',
        'R55W-9' : 'Excavator(4.5~6t)',
        'R110-7' : 'Excavator(7~11t)',
        'R110-7A' : 'Excavator(7~11t)',
        'R140LC-7' : 'Excavator(12~15.5t)',
        'R140LC-7A' : 'Excavator(12~15.5t)',
        'R140LC-9' : 'Excavator(12~15.5t)',
        'R145CR-9' : 'Excavator(12~15.5t)',
        'R15-7' : 'Mini Excavator(0.1~1.6t)',
        'R160LC-3' : 'Excavator(16~19t)',
        'R160LC-7' : 'Excavator(16~19t)',
        'R160LC-7A' : 'Excavator(16~19t)',
        'R160LC-9' : 'Excavator(16~19t)',
        'R16-7' : 'Mini Excavator(0.1~1.6t)',
        'R16-9' : 'Mini Excavator(0.1~1.6t)',
        'R180LC-7' : 'Excavator(16~19t)',
        'R180LC-7A' : 'Excavator(16~19t)',
        'R180LC-9' : 'Excavator(16~19t)',
        'R210LC-7' : 'Excavator(20~27t)',
        'R210LC-7A' : 'Excavator(20~27t)',
        'R210LC-7H' : 'Excavator(20~27t)',
        'R210LC-9' : 'Excavator(20~27t)',
        'R210NLC-7' : 'Excavator(20~27t)',
        'R210NLC-7A' : 'Excavator(20~27t)',
        'R210NLC-9' : 'Excavator(20~27t)',
        'R22-7' : 'Mini Excavator(1.7~2.5t)',
        'R235LCR-9' : 'Excavator(20~27t)',
        'R250LC-7' : 'Excavator(20~27t)',
        'R250LC-7A' : 'Excavator(20~27t)',
        'R250LC-9' : 'Excavator(20~27t)',
        'R27Z-9' : 'Mini Excavator(2.6~4t)',
        'R28-7' : 'Mini Excavator(2.6~4t)',
        'R290LC-7' : 'Excavator(28~32t)',
        'R290LC-7A' : 'Excavator(28~32t)',
        'R290LC-7H' : 'Excavator(28~32t)',
        'R290LC-9' : 'Excavator(28~32t)',
        'R300LC-7' : 'Excavator(28~32t)',
        'R305LC-7' : 'Excavator(28~32t)',
        'R320LC-7' : 'Excavator(28~32t)',
        'R320LC-7A' : 'Excavator(28~32t)',
        'R320LC-9' : 'Excavator(28~32t)',
        'R35-7' : 'Mini Excavator(2.6~4t)',
        'R35Z-7' : 'Mini Excavator(2.6~4t)',
        'R35Z-7A' : 'Mini Excavator(2.6~4t)',
        'R35Z-9' : 'Mini Excavator(2.6~4t)',
        'R360LC-7' : 'Excavator(33t over)',
        'R360LC-7A' : 'Excavator(33t over)',
        'R360LC-9' : 'Excavator(33t over)',
        'R36N-7' : 'Mini Excavator(2.6~4t)',
        'R370LC-7' : 'Excavator(33t over)',
        'R450LC-7' : 'Excavator(33t over)',
        'R450LC-7A' : 'Excavator(33t over)',
        'R480LC-9' : 'Excavator(33t over)',
        'R500LC-7' : 'Excavator(33t over)',
        'R500LC-7A' : 'Excavator(33t over)',
        'R520LC-9' : 'Excavator(33t over)',
        'R55-7' : 'Excavator(4.5~6t)',
        'R55-7A' : 'Excavator(4.5~6t)',
        'R55-9' : 'Excavator(4.5~6t)',
        'R75-7' : 'Excavator(7~11t)',
        'R800LC-7A' : 'Excavator(33t over)',
        'R80-7' : 'Excavator(7~11t)',
        'R80-7A' : 'Excavator(7~11t)',
        'R210LC-7' : 'Excavator(20~27t)',
        'R210LC-7H' : 'Excavator(20~27t)',
        'R210-7' : 'Excavator(20~27t)',
        'R220LC-7' : 'Excavator(20~27t)',
        'R220LC-7H' : 'Excavator(20~27t)',
        'E60' : 'Excavator(4.5~6t)',
        'E80' : 'Excavator(7~11t)',
        'TXC140-1' : 'Excavator(12~15.5t)',
        'TXC140LC-2' : 'Excavator(12~15.5t)',
        'TXC175-1' : 'Excavator(16~19t)',
        'TXC180LC-2' : 'Excavator(16~19t)',
        'TXC225-1' : 'Excavator(20~27t)',
        'TXC225LC-2' : 'Excavator(20~27t)',
        'TXC255-1' : 'Excavator(20~27t)',
        'TXC255LC-2' : 'Excavator(20~27t)',
        'TXC300-1' : 'Excavator(28~32t)',
        'TXC300LC-2' : 'Excavator(28~32t)',
        'TXC340-1' : 'Excavator(33t over)',
        'TXC340LC-2' : 'Excavator(33t over)',
        'TXC420-1' : 'Excavator(33t over)',
        'TXC420LC-2' : 'Excavator(33t over)',
        'TXC470-1' : 'Excavator(33t over)',
        'TXC480LC-2' : 'Excavator(33t over)',
        'TXC500-1' : 'Excavator(33t over)',
        'TXC520LC-2' : 'Excavator(33t over)',
        'DH150LC-7 (ERP)' : 'Excavator(12~15.5t)',
        'DH80 GOLD (ERP)' : 'Excavator(7~11t)',
        'DX140LC' : 'Excavator(12~15.5t)',
        'DX140LC' : 'Excavator(12~15.5t)',
        'DX140LCR' : 'Excavator(12~15.5t)',
        'DX15' : 'Mini Excavator(0.1~1.6t)',
        'DX160LC' : 'Excavator(16~19t)',
        'DX18' : 'Mini Excavator(1.7~2.5t)',
        'DX180LC' : 'Mini Excavator(1.7~2.5t)',
        'DX180LC' : 'Mini Excavator(1.7~2.5t)',
        'DX220LC (DTC)' : 'Excavator(20~27t)',
        'DX225LC' : 'Excavator(20~27t)',
        'DX225LC' : 'Excavator(20~27t)',
        'DX225LC (DL06 ENGINE)' : 'Excavator(20~27t)',
        'DX225LC (DTC)' : 'Excavator(20~27t)',
        'DX225LCA' : 'Excavator(20~27t)',
        'DX225NLC' : 'Excavator(20~27t)',
        'DX230LC' : 'Excavator(20~27t)',
        'DX255LC' : 'Excavator(20~27t)',
        'DX255LC (ERP)' : 'Excavator(20~27t)',
        'DX27Z' : 'Mini Excavator(2.6~4t)',
        'DX300LC' : 'Excavator(28~32t)',
        'DX300LC' : 'Excavator(28~32t)',
        'DX300LC-DEMOLITION' : 'Excavator(28~32t)',
        'DX300LC-DEMOLITION' : 'Excavator(28~32t)',
        'DX30Z' : 'Excavator(28~32t)',
        'DX340LC' : 'Excavator(33t over)',
        'DX350LC' : 'Excavator(33t over)',
        'DX35Z' : 'Mini Excavator(2.6~4t)',
        'DX380LC' : 'Excavator(33t over)',
        'DX420LC' : 'Excavator(33t over)',
        'DX420LC-DEMOLITION' : 'Excavator(33t over)',
        'DX420LC-DEMOLITION' : 'Excavator(33t over)',
        'DX480LC' : 'Excavator(33t over)',
        'DX520LC' : 'Excavator(33t over)',
        'DX55 E' : 'Excavator(4.5~6t)',
        'DX60R' : 'Excavator(4.5~6t)',
        'DX80R' : 'Excavator(7~11t)',
        'SOLAR 010' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 010' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 015' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 015 PLUS' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 015 PLUS' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 015 PLUS' : 'Mini Excavator(0.1~1.6t)',
        'SOLAR 018-VT' : 'Mini Excavator(1.7~2.5t)',
        'SOLAR 018-VT' : 'Mini Excavator(1.7~2.5t)',
        'SOLAR 030' : 'Mini Excavator(2.6~4t)',
        'SOLAR 030 PLUS' : 'Mini Excavator(2.6~4t)',
        'SOLAR 030 PLUS' : 'Mini Excavator(2.6~4t)',
        'SOLAR 035' : 'Mini Excavator(2.6~4t)',
        'SOLAR 035' : 'Mini Excavator(2.6~4t)',
        'SOLAR 130-III' : 'Excavator(12~15.5t)',
        'SOLAR 130LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 130LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 140LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 140LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 150LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 155LC-V' : 'Excavator(12~15.5t)',
        'SOLAR 170-III' : 'Excavator(16~19t)',
        'SOLAR 170LC-V' : 'Excavator(16~19t)',
        'SOLAR 170LC-V' : 'Excavator(16~19t)',
        'SOLAR 175LC-V' : 'Excavator(16~19t)',
        'SOLAR 175LC-V' : 'Excavator(16~19t)',
        'SOLAR 220LC-6' : 'Excavator(20~27t)',
        'SOLAR 220LC-III' : 'Excavator(20~27t)',
        'SOLAR 220LC-V' : 'Excavator(20~27t)',
        'SOLAR 220LC-V' : 'Excavator(20~27t)',
        'SOLAR 220N-V' : 'Excavator(20~27t)',
        'SOLAR 225LC-7A' : 'Excavator(20~27t)',
        'SOLAR 225LC-V' : 'Excavator(20~27t)',
        'SOLAR 225LC-V' : 'Excavator(20~27t)',
        'SOLAR 225NLC-V' : 'Excavator(20~27t)',
        'SOLAR 230LC-V' : 'Excavator(20~27t)',
        'SOLAR 250LC-V' : 'Excavator(20~27t)',
        'SOLAR 250LC-V' : 'Excavator(20~27t)',
        'SOLAR 255LC-V' : 'Excavator(20~27t)',
        'SOLAR 255LC-V' : 'Excavator(20~27t)',
        'SOLAR 280LC-III' : 'Excavator(28~32t)',
        'SOLAR 290LC-V' : 'Excavator(28~32t)',
        'SOLAR 290LC-V' : 'Excavator(28~32t)',
        'SOLAR 300LC-7A' : 'Excavator(28~32t)',
        'SOLAR 300LC-V' : 'Excavator(28~32t)',
        'SOLAR 330-III' : 'Excavator(33t over)',
        'SOLAR 330LC-V' : 'Excavator(33t over)',
        'SOLAR 340LC-7' : 'Excavator(33t over)',
        'SOLAR 340LC-V' : 'Excavator(33t over)',
        'SOLAR 400LC-III' : 'Excavator(33t over)',
        'SOLAR 400LC-V' : 'Excavator(33t over)',
        'SOLAR 420LC-V' : 'Excavator(33t over)',
        'SOLAR 450-III' : 'Excavator(33t over)',
        'SOLAR 450LC-V' : 'Excavator(33t over)',
        'SOLAR 470LC-V' : 'Excavator(33t over)',
        'SOLAR 500LC-V' : 'Excavator(33t over)',
        'SOLAR 55-V' : 'Excavator(4.5~6t)',
        'SOLAR 55-V PLUS' : 'Excavator(4.5~6t)',
        'SOLAR 70-III' : 'Excavator(7~11t)',
        'SOLAR 75-V' : 'Excavator(7~11t)',
        'SOLAR 75-V' : 'Excavator(7~11t)',
        'B55W-2' : 'Excavator(4.5~6t)',
        'DX140W' : 'Excavator(12~15.5t)',
        'DX140W TIER-II' : 'Excavator(12~15.5t)',
        'DX140W TIER-III' : 'Excavator(12~15.5t)',
        'DX160W' : 'Excavator(16~19t)',
        'DX160W RAIL WAY' : 'Excavator(16~19t)',
        'DX170W' : 'Excavator(16~19t)',
        'DX190W' : 'Excavator(16~19t)',
        'DX190W MATERIAL' : 'Excavator(16~19t)',
        'DX190W MATERIAL' : 'Excavator(16~19t)',
        'DX210W' : 'Excavator(20~27t)',
        'DX210W MATERIAL' : 'Excavator(20~27t)',
        'DX210W MATERIAL' : 'Excavator(20~27t)',
        'DX53W' : 'Excavator(4.5~6t)',
        'DX55W' : 'Excavator(4.5~6t)',
        'DX60W-E' : 'Excavator(4.5~6t)',
        'SOLAR 130W-III' : 'Excavator(12~15.5t)',
        'SOLAR 130W-V(1)' : 'Excavator(12~15.5t)',
        'SOLAR 130W-V(2)' : 'Excavator(12~15.5t)',
        'SOLAR 140W-V' : 'Excavator(12~15.5t)',
        'SOLAR 140W-V' : 'Excavator(12~15.5t)',
        'SOLAR 140W-V RAIL WAY' : 'Excavator(12~15.5t)',
        'SOLAR 160W-V' : 'Excavator(16~19t)',
        'SOLAR 170W-III' : 'Excavator(16~19t)',
        'SOLAR 170W-V' : 'Excavator(16~19t)',
        'SOLAR 180W-V' : 'Excavator(16~19t)',
        'SOLAR 185W-V' : 'Excavator(16~19t)',
        'SOLAR 200W-III' : 'Excavator(20~27t)',
        'SOLAR 200W-V' : 'Excavator(20~27t)',
        'SOLAR 210W-V' : 'Excavator(20~27t)',
        'SOLAR 210W-V' : 'Excavator(20~27t)',
        'SOLAR 55W-V' : 'Excavator(4.5~6t)',
        'SOLAR 55W-V SUPER' : 'Excavator(4.5~6t)',
        'SOLAR 55W-V SUPER' : 'Excavator(4.5~6t)',
        'S130LC-V' : 'Excavator(12~15.5t)',
        'S130LC-V' : 'Excavator(12~15.5t)',
        'S140LC-V' : 'Excavator(12~15.5t)',
        'S140LC-V' : 'Excavator(12~15.5t)',
        'S150LC-V' : 'Excavator(12~15.5t)',
        'S155LC-V' : 'Excavator(12~15.5t)',
        'S170LC-V' : 'Excavator(16~19t)',
        'S170LC-V' : 'Excavator(16~19t)',
        'S175LC-V' : 'Excavator(16~19t)',
        'S175LC-V' : 'Excavator(16~19t)',
        'S220LC-6' : 'Excavator(20~27t)',
        'S220LC-V' : 'Excavator(20~27t)',
        'S220LC-V' : 'Excavator(20~27t)',
        'S220N-V' : 'Excavator(20~27t)',
        'S225LC-7A' : 'Excavator(20~27t)',
        'S225LC-V' : 'Excavator(20~27t)',
        'S225LC-V' : 'Excavator(20~27t)',
        'S225NLC-V' : 'Excavator(20~27t)',
        'S230LC-V' : 'Excavator(20~27t)',
        'S250LC-V' : 'Excavator(20~27t)',
        'S250LC-V' : 'Excavator(20~27t)',
        'S255LC-V' : 'Excavator(20~27t)',
        'S255LC-V' : 'Excavator(20~27t)',
        'S290LC-V' : 'Excavator(28~32t)',
        'S290LC-V' : 'Excavator(28~32t)',
        'S300LC-7A' : 'Excavator(28~32t)',
        'S300LC-V' : 'Excavator(28~32t)',
        'S330LC-V' : 'Excavator(33t over)',
        'S340LC-7' : 'Excavator(33t over)',
        'S340LC-V' : 'Excavator(33t over)',
        'S400LC-V' : 'Excavator(33t over)',
        'S420LC-V' : 'Excavator(33t over)',
        'S450LC-V' : 'Excavator(33t over)',
        'S470LC-V' : 'Excavator(33t over)',
        'S500LC-V' : 'Excavator(33t over)',
        'S80' : 'Excavator(7~11t)',
        'S010' : 'Mini Excavator(0.1~1.6t)',
        'S010' : 'Mini Excavator(0.1~1.6t)',
        'S015' : 'Mini Excavator(0.1~1.6t)',
        'S015P' : 'Mini Excavator(0.1~1.6t)',
        'S015P' : 'Mini Excavator(0.1~1.6t)',
        'S015P' : 'Mini Excavator(0.1~1.6t)',
        'S030' : 'Mini Excavator(2.6~4t)',
        'S030P' : 'Mini Excavator(2.6~4t)',
        'S030P' : 'Mini Excavator(2.6~4t)',
        'S035' : 'Mini Excavator(2.6~4t)',
        'S035' : 'Mini Excavator(2.6~4t)',
        'S55' : 'Excavator(4.5~6t)',
        'S55-VP' : 'Excavator(4.5~6t)',
        'S75-V' : 'Excavator(7~11t)',
        'S75-V' : 'Excavator(7~11t)',
        'DX15' : 'Mini Excavator(0.1~1.6t)',
        'DX27Z' : 'Mini Excavator(2.6~4t)',
        'DX30Z' : 'Mini Excavator(2.6~4t)',
        'DX35Z' : 'Mini Excavator(2.6~4t)',
        'DX55' : 'Excavator(4.5~6t)',
        'DX60R' : 'Excavator(4.5~6t)',
        'DX80R' : 'Excavator(7~11t)',
        'S130-3' : 'Excavator(12~15.5t)',
        'S170-3' : 'Excavator(16~19t)',
        'S220LC-3' : 'Excavator(20~27t)',
        'S280LC-3' : 'Excavator(28~32t)',
        'S330-3' : 'Excavator(33t over)',
        'S400LC-3' : 'Excavator(33t over)',
        'S450-3' : 'Excavator(33t over)',
        'S70-3' : 'Excavator(7~11t)',
        'DX140LC' : 'Excavator(12~15.5t)',
        'DX140LC' : 'Excavator(12~15.5t)',
        'DX140LCR' : 'Excavator(12~15.5t)',
        'DX160LC' : 'Excavator(16~19t)',
        'DX180LC' : 'Excavator(16~19t)',
        'DX180LC' : 'Excavator(16~19t)',
        'DX225LC' : 'Excavator(20~27t)',
        'DX225LC' : 'Excavator(20~27t)',
        'DX225LC' : 'Excavator(20~27t)',
        'DX225LCA' : 'Excavator(20~27t)',
        'DX225NLC' : 'Excavator(20~27t)',
        'DX230LC' : 'Excavator(20~27t)',
        'DX255LC' : 'Excavator(20~27t)',
        'DX255LC' : 'Excavator(20~27t)',
        'DX300LC' : 'Excavator(28~32t)',
        'DX300LC' : 'Excavator(28~32t)',
        'DX300LCA' : 'Excavator(28~32t)',
        'DX300LC-DEMOLITION' : 'Excavator(28~32t)',
        'DX340LC' : 'Excavator(33t over)',
        'DX350LC' : 'Excavator(33t over)',
        'DX380LC' : 'Excavator(33t over)',
        'DX420LC' : 'Excavator(33t over)',
        'DX420LC-DEOMOLITION' : 'Excavator(33t over)',
        'DX-480LC' : 'Excavator(33t over)',
        'DX520LC' : 'Excavator(33t over)',
        'S150LC-7A' : 'Excavator(12~15.5t)',
        'S130W-3' : 'Excavator(12~15.5t)',
        'S170W-3' : 'Excavator(16~19t)',
        'S200W-3' : 'Excavator(20~27t)',
        'B55W-1' : 'Excavator(4.5~6t)',
        'S130W-V' : 'Excavator(12~15.5t)',
        'S130W-V' : 'Excavator(12~15.5t)',
        'S140W-V' : 'Excavator(12~15.5t)',
        'S140W-V' : 'Excavator(12~15.5t)',
        'S140W-VRAILWAY' : 'Excavator(12~15.5t)',
        'S160W-V' : 'Excavator(16~19t)',
        'S170W-V' : 'Excavator(16~19t)',
        'S180W-V' : 'Excavator(16~19t)',
        'S185W-V' : 'Excavator(16~19t)',
        'S200W-V' : 'Excavator(20~27t)',
        'S210W-V' : 'Excavator(20~27t)',
        'S210W-V' : 'Excavator(20~27t)',
        'S55W-V' : 'Excavator(4.5~6t)',
        'S55W-VP' : 'Excavator(4.5~6t)',
        'B55W-2' : 'Excavator(4.5~6t)',
        'DX140WTIER-2' : 'Excavator(12~15.5t)',
        'DX140WTIER-3' : 'Excavator(12~15.5t)',
        'DX160WRAILWAY' : 'Excavator(16~19t)',
        'DX170W' : 'Excavator(16~19t)',
        'DX190W' : 'Excavator(16~19t)',
        'DX190WMATERIAL' : 'Excavator(16~19t)',
        'DX210W' : 'Excavator(20~27t)',
        'DX210WMATERAIL' : 'Excavator(20~27t)',
        'DX53W' : 'Excavator(4.5~6t)',
        'DX55W' : 'Excavator(4.5~6t)',
        'E55W' : 'Excavator(4.5~6t)',
        'E60' : 'Excavator(4.5~6t)',
        'E80' : 'Excavator(7~11t)',
        'DX27Z' : 'Mini Excavator(2.6~4t)',
        'DX30Z' : 'Mini Excavator(2.6~4t)',
        'DX35Z' : 'Mini Excavator(2.6~4t)',
        'DX55A' : 'Excavator(4.5~6t)',
        'DX55MTA' : 'Excavator(4.5~6t)',
        'DX60R' : 'Excavator(4.5~6t)',
        'DX140A' : 'Excavator(12~15.5t)',
        'DX140R' : 'Excavator(12~15.5t)',
        'DX220LCA' : 'Excavator(20~27t)',
        'DX300LCA' : 'Excavator(28~32t)',
        'DX360LCA' : 'Excavator(33t over)',
        'DX380LCA' : 'Excavator(33t over)',
        'DX420LCA' : 'Excavator(33t over)',
        'DX480LCA' : 'Excavator(33t over)',
        'DX500LCA' : 'Excavator(33t over)',
        'DX520LCA' : 'Excavator(33t over)',
        'DX700LCA' : 'Excavator(33t over)',
        'DX55WA' : 'Excavator(4.5~6t)',
        'DX140WA' : 'Excavator(12~15.5t)',
        'DX210WA' : 'Excavator(20~27t)',
        'DX27Z' : 'Mini Excavator(2.6~4t)',
        'DX30Z' : 'Mini Excavator(2.6~4t)',
        'DX35Z' : 'Mini Excavator(2.6~4t)',
        'DX60R' : 'Excavator(4.5~6t)',
        'DX15' : 'Mini Excavator(0.1~1.6t)',
        'DX55' : 'Excavator(4.5~6t)',
        'DX55MT' : 'Excavator(4.5~6t)',
        'DX140' : 'Excavator(12~15.5t)',
        'DX220LC' : 'Excavator(20~27t)',
        'DX300LCA' : 'Excavator(28~32t)',
        'DX350LC' : 'Excavator(33t over)',
        'DX360LCA' : 'Excavator(33t over)',
        'DX380LC' : 'Excavator(33t over)',
        'DX420LCA' : 'Excavator(33t over)',
        'DX480LCA' : 'Excavator(33t over)',
        'DX500LCA' : 'Excavator(33t over)',
        'DX520LCA' : 'Excavator(33t over)',
        'DX55W' : 'Excavator(4.5~6t)',
        'DX140W' : 'Excavator(12~15.5t)',
        'DX210W' : 'Excavator(20~27t)',
        'S255LC-V' : 'Excavator(20~27t)',
        'S300LC-7' : 'Excavator(28~32t)',
        'S420LC-V' : 'Excavator(33t over)',
        'S470LC-V' : 'Excavator(33t over)',
        'S500LC-V' : 'Excavator(33t over)',
        'DX140LC' : 'Excavator(12~15.5t)',
        'DX140LCR' : 'Excavator(12~15.5t)',
        'DX225LCA' : 'Excavator(20~27t)',
        'DX225NLC' : 'Excavator(20~27t)',
        'DX225SLR' : 'Excavator(20~27t)',
        'DX255LC' : 'Excavator(20~27t)',
        'DX300LCA' : 'Excavator(28~32t)',
        'DX300SLR' : 'Excavator(28~32t)',
        'DX340LC' : 'Excavator(33t over)',
        'DX420LCA' : 'Excavator(33t over)',
        'DX480LCA' : 'Excavator(33t over)',
        'DX520LC' : 'Excavator(33t over)',
        'DX300LL' : 'Excavator(28~32t)',
        'DX300LC' : 'Excavator(28~32t)',
        'DX700LC' : 'Excavator(33t over)',
        'DX53W' : 'Excavator(4.5~6t)',
        'DX55W' : 'Excavator(4.5~6t)',
        'S180W-V' : 'Excavator(16~19t)',
        'S210W-V' : 'Excavator(20~27t)',
        'DX140W' : 'Excavator(12~15.5t)',
        'DX190W' : 'Excavator(16~19t)',
        'DX210W' : 'Excavator(20~27t)',
        'DX27Z' : 'Mini Excavator(2.6~4t)',
        'DX30Z' : 'Mini Excavator(2.6~4t)',
        'DX35Z' : 'Mini Excavator(2.6~4t)',
        'DX55' : 'Excavator(4.5~6t)',
        'DX60R' : 'Excavator(4.5~6t)',
        'DX80R' : 'Excavator(7~11t)',
        'S007' : 'Mini Excavator(0.1~1.6t)',
        'S010' : 'Mini Excavator(0.1~1.6t)',
        'S015' : 'Mini Excavator(0.1~1.6t)',
        'S30¡¡¡¡¡¡' : 'Mini Excavator(2.6~4t)',
        'S030' : 'Mini Excavator(2.6~4t)',
        'DH012' : 'Excavator(4.5~6t)',
        'S035¡¡¡¡¡¡' : 'Mini Excavator(2.6~4t)',
        'DH016W' : 'Excavator(4.5~6t)',
        'DH02' : 'Excavator(4.5~6t)',
        'DH02S' : 'Excavator(4.5~6t)',
        'S50W' : 'Excavator(4.5~6t)',
        'DH028' : 'Excavator(4.5~6t)',
        'DH028-2' : 'Excavator(4.5~6t)',
        'S50W-3' : 'Excavator(4.5~6t)',
        'S50¡¡¡¡¡¡¡¡' : 'Excavator(4.5~6t)',
        'S55W-V' : 'Excavator(4.5~6t)',
        'S55-V' : 'Excavator(4.5~6t)',
        'S70-3' : 'Excavator(7~11t)',
        'S80W' : 'Excavator(7~11t)',
        'S80W-3' : 'Excavator(7~11t)',
        'DH03' : 'Excavator(4.5~6t)',
        'DH05W' : 'Excavator(4.5~6t)',
        'DH04' : 'Mini Excavator(2.6~4t)',
        'DH110' : 'Excavator(7~11t)',
        'S130W' : 'Excavator(12~15.5t)',
        'S120W' : 'Excavator(12~15.5t)',
        'S130W-3' : 'Excavator(12~15.5t)',
        'DH04M' : 'Mini Excavator(2.6~4t)',
        'S130W-V' : 'Excavator(12~15.5t)',
        'S130' : 'Excavator(12~15.5t)',
        'S130-3' : 'Excavator(12~15.5t)',
        'S130-2' : 'Excavator(12~15.5t)',
        'S130-V' : 'Excavator(12~15.5t)',
        'S130LC-V' : 'Excavator(12~15.5t)',
        'S150M' : 'Excavator(12~15.5t)',
        'DH07W' : 'Excavator(7~11t)',
        'S170' : 'Excavator(16~19t)',
        'S170-3' : 'Excavator(16~19t)',
        'S170W-3' : 'Excavator(16~19t)',
        'S170W-V' : 'Excavator(16~19t)',
        'DH07-3' : 'Excavator(7~11t)',
        'DH08' : 'Excavator(7~11t)',
        'DH07' : 'Excavator(7~11t)',
        'S200W' : 'Excavator(20~27t)',
        'S220LC' : 'Excavator(20~27t)',
        'S200W-3' : 'Excavator(20~27t)',
        'S200W-V' : 'Excavator(20~27t)',
        'S220LC-3' : 'Excavator(20~27t)',
        'S220LC-V' : 'Excavator(20~27t)',
        'S220LC' : 'Excavator(20~27t)',
        'DH10' : 'Mini Excavator(0.1~1.6t)',
        'DH10L' : 'Mini Excavator(0.1~1.6t)',
        'S280¡¡¡¡¡¡' : 'Excavator(28~32t)',
        'S280LC-3' : 'Excavator(28~32t)',
        'S290LC-V' : 'Excavator(28~32t)',
        'S320' : 'Excavator(28~32t)',
        'S330-3' : 'Excavator(33t over)',
        'S330LC-V' : 'Excavator(33t over)',
        'S360LC-3' : 'Excavator(33t over)',
        'S360LC-V' : 'Excavator(33t over)',
        'S400LC-V' : 'Excavator(33t over)',
        'DH16' : 'Excavator(4.5~6t)',
        'S450-3' : 'Excavator(33t over)',
        'S450' : 'Excavator(33t over)',
        'S450LC-V' : 'Excavator(33t over)',
        'DH20' : 'Excavator(20~27t)',
        'RH30D' : 'Excavator(28~32t)',
        'S400LC-3' : 'Excavator(33t over)',
        'S010' : 'Mini Excavator(0.1~1.6t)',
        'S015' : 'Mini Excavator(0.1~1.6t)',
        'S030' : 'Mini Excavator(2.6~4t)',
        'S035' : 'Mini Excavator(2.6~4t)',
        'S50' : 'Excavator(4.5~6t)',
        'S50w-3' : 'Excavator(4.5~6t)',
        'S50w-¥´' : 'Excavator(4.5~6t)',
        'S55-¥´' : 'Excavator(4.5~6t)',
        'S55¥´-G' : 'Excavator(4.5~6t)',
        'S70-3' : 'Excavator(7~11t)',
        'S80W-3' : 'Excavator(7~11t)',
        'S130-V' : 'Excavator(12~15.5t)',
        'S130W-3A' : 'Excavator(12~15.5t)',
        'S130W-3B' : 'Excavator(12~15.5t)',
        'S170-3' : 'Excavator(16~19t)',
        'S170W-3' : 'Excavator(16~19t)',
        'S200W-3' : 'Excavator(20~27t)',
        'S200W-¥´' : 'Excavator(20~27t)',
        'S220LC-3' : 'Excavator(20~27t)',
        'S220LC-¥´' : 'Excavator(20~27t)',
        'S280LC-3a' : 'Excavator(28~32t)',
        'S280LC-3b' : 'Excavator(28~32t)',
        'S290LC-V' : 'Excavator(28~32t)',
        'S330LC-3' : 'Excavator(33t over)',
        'S330LC-V' : 'Excavator(33t over)',
        'S360LC-3' : 'Excavator(33t over)',
        'S360LC-V' : 'Excavator(33t over)',
        'S400LC-V' : 'Excavator(33t over)',
        'S400LC-3' : 'Excavator(33t over)',
        'S450LC-3' : 'Excavator(33t over)',
        'S450LC-V' : 'Excavator(33t over)',
        'S200W-V' : 'Excavator(20~27t)',
        'S170W-V' : 'Excavator(16~19t)',
        'S130W-V' : 'Excavator(12~15.5t)',
        'S55W-V' : 'Excavator(4.5~6t)',
        'R16' : 'Mini Excavator(0.1~1.6t)',
        'R27Z' : 'Mini Excavator(2.6~4t)',
        'R35Z' : 'Mini Excavator(2.6~4t)',
        'R55i' : 'Excavator(4.5~6t)',
        'R80CR' : 'Excavator(7~11t)',
        'R140LC' : 'Excavator(12~15.5t)',
        'R145LCR' : 'Excavator(12~15.5t)',
        'R220LC' : 'Excavator(20~27t)',
        'R235LCR' : 'Excavator(20~27t)',
        'R300LC' : 'Excavator(28~32t)',
        'R300LC-E' : 'Excavator(28~32t)',
        'R380LC' : 'Excavator(33t over)',
        'R480LC' : 'Excavator(33t over)',
        'R520LC' : 'Excavator(33t over)',
        'R8000LC-7A' : 'Excavator(33t over)',
        'R60CR' : 'Excavator(4.5~6t)',
        'R55wi' : 'Excavator(4.5~6t)',
        'R140W' : 'Excavator(12~15.5t)',
        'R210W' : 'Excavator(20~27t)',
        'R140LC-9' : 'Excavator(12~15.5t)',
        'R145CR-9' : 'Excavator(12~15.5t)',
        'R160LC-9' : 'Excavator(16~19t)',
        'R180LC-9' : 'Excavator(16~19t)',
        'R210LC-9' : 'Excavator(20~27t)',
        'R235LCR-9' : 'Excavator(20~27t)',
        'R250LC-9' : 'Excavator(20~27t)',
        'R290LC-9' : 'Excavator(28~32t)',
        'R320LC-9' : 'Excavator(28~32t)',
        'R380LC-9' : 'Excavator(33t over)',
        'R480LC-9' : 'Excavator(33t over)',
        'R520LC-9' : 'Excavator(33t over)',
        'R800LC-9' : 'Excavator(33t over)',
        'R1200-9' : 'Excavator(12~15.5t)',
        'R140LC-9S' : 'Excavator(12~15.5t)',
        'R160LC-9S' : 'Excavator(16~19t)',
        'R180LC-9S' : 'Excavator(16~19t)',
        'R220LC-9S' : 'Excavator(20~27t)',
        'R220LC-9SLR' : 'Excavator(20~27t)',
        'R220LC-9SH' : 'Excavator(20~27t)',
        'R260LC-9S' : 'Excavator(20~27t)',
        'R300LC-9S' : 'Excavator(28~32t)',
        'R300LC-9SLR' : 'Excavator(28~32t)',
        'R300LC-9SH' : 'Excavator(28~32t)',
        'R330LC-9S' : 'Excavator(33t over)',
        'R330LC-9SH' : 'Excavator(33t over)',
        'R380LC-9SH' : 'Excavator(33t over)',
        'R480LC-9S' : 'Excavator(33t over)',
        'R520LC-9S' : 'Excavator(33t over)',
        'R110-7' : 'Excavator(7~11t)',
        'R110M-7' : 'Excavator(7~11t)',
        'R140LC-7' : 'Excavator(12~15.5t)',
        'R140LCD-7' : 'Excavator(12~15.5t)',
        'R140LCM-7' : 'Excavator(12~15.5t)',
        'R160LC-7' : 'Excavator(16~19t)',
        'R180LC-7' : 'Excavator(16~19t)',
        'R210LC-7' : 'Excavator(20~27t)',
        'R210LC-7H' : 'Excavator(20~27t)',
        'R220-7H' : 'Excavator(20~27t)',
        'R220LC-7H' : 'Excavator(20~27t)',
        'R250LC-7' : 'Excavator(20~27t)',
        'R305LC-7' : 'Excavator(28~32t)',
        'R320LC-7' : 'Excavator(28~32t)',
        'R370LC-7' : 'Excavator(33t over)',
        'R450LC-7' : 'Excavator(33t over)',
        'R500LC-7' : 'Excavator(33t over)',
        'R110-7A' : 'Excavator(7~11t)',
        'R140LC-7A' : 'Excavator(12~15.5t)',
        'R160LC-7A' : 'Excavator(16~19t)',
        'R180LC-7A' : 'Excavator(16~19t)',
        'R210LC-7A' : 'Excavator(20~27t)',
        'R210NLC-7A' : 'Excavator(20~27t)',
        'R210LC-7ALR' : 'Excavator(20~27t)',
        'R250LC-7A' : 'Excavator(20~27t)',
        'R290LC-7A' : 'Excavator(28~32t)',
        'R320LC-7A' : 'Excavator(28~32t)',
        'R290LC-7ALR' : 'Excavator(28~32t)',
        'R360LC-7A' : 'Excavator(33t over)',
        'R450LC-7A' : 'Excavator(33t over)',
        'R500LC-7A' : 'Excavator(33t over)',
        'R800LC-7A' : 'Excavator(33t over)',
        'R55W-9' : 'Excavator(4.5~6t)',
        'R140W-9' : 'Excavator(12~15.5t)',
        'R170W-9' : 'Excavator(16~19t)',
        'R210W-9' : 'Excavator(20~27t)',
        'R210W-9MH' : 'Excavator(20~27t)',
        'R140W-9S' : 'Excavator(12~15.5t)',
        'R170W-9S' : 'Excavator(16~19t)',
        'R210W-9S' : 'Excavator(20~27t)',
        'R140W-7' : 'Excavator(12~15.5t)',
        'R170W-7' : 'Excavator(16~19t)',
        'R200W-7' : 'Excavator(20~27t)',
        'R140W-7A' : 'Excavator(12~15.5t)',
        'R170W-7A' : 'Excavator(16~19t)',
        'R200W-7A' : 'Excavator(20~27t)',
        'R16-9' : 'Mini Excavator(0.1~1.6t)',
        'R27Z-9' : 'Mini Excavator(2.6~4t)',
        'R35Z-9' : 'Mini Excavator(2.6~4t)',
        'R60CR-9' : 'Excavator(4.5~6t)',
        'R80CR-9' : 'Excavator(7~11t)',
        'R55-9' : 'Excavator(4.5~6t)',
        'R60-9S' : 'Excavator(4.5~6t)',
        'R60W-9S' : 'Excavator(4.5~6t)',
        'R35Z-7' : 'Mini Excavator(2.6~4t)',
        'R55-7' : 'Excavator(4.5~6t)',
        'R55W-7' : 'Excavator(4.5~6t)',
        'R80-7' : 'Excavator(7~11t)',
        'R35Z-7A' : 'Mini Excavator(2.6~4t)',
        'R55-7A' : 'Excavator(4.5~6t)',
        'R55W-7A' : 'Excavator(4.5~6t)',
        'R80-7A' : 'Excavator(7~11t)',
        'R35Z-7' : 'Mini Excavator(2.6~4t)',
        'R55' : 'Excavator(4.5~6t)',
        'R55W' : 'Excavator(4.5~6t)',
        'HX15SR' : 'Mini Excavator(0.1~1.6t)',
        'HX28S' : 'Mini Excavator(2.6~4t)',
        'R450' : 'Excavator(4.5~6t)',
        'R500' : 'Excavator(4.5~6t)',
        'R500W' : 'Excavator(4.5~6t)',
        'HX600W' : 'Excavator(4.5~6t)',
        'HX60W-2' : 'Excavator(4.5~6t)',
        'R1300W' : 'Excavator(12~15.5t)',
        'R1700W-3' : 'Excavator(16~19t)',
        'R2000W' : 'Excavator(20~27t)',
        'R2000W-2' : 'Excavator(20~27t)',
        'R1300LC' : 'Excavator(12~15.5t)',
        'R1300LC-3' : 'Excavator(12~15.5t)',
        'R1800LC-3' : 'Excavator(16~19t)',
        'HX80' : 'Excavator(7~11t)',
        'R2000LC' : 'Excavator(20~27t)',
        'R2000LC-2' : 'Excavator(20~27t)',
        'R2200LC-3' : 'Excavator(20~27t)',
        'HX100' : 'Excavator(7~11t)',
        'R2800LC' : 'Excavator(28~32t)',
        'R2800K' : 'Excavator(28~32t)',
        'R2800KLC' : 'Excavator(28~32t)',
        'R2900LC-3' : 'Excavator(28~32t)',
        'R3000' : 'Excavator(28~32t)',
        'R3200LC' : 'Excavator(28~32t)',
        'R3600LC-3' : 'Excavator(33t over)',
        'R4200' : 'Excavator(33t over)',
        'R4500LC' : 'Excavator(33t over)',
        'R4500LC-7' : 'Excavator(33t over)',
        'R4500LC-3' : 'Excavator(33t over)',
        'R4500LC-3A' : 'Excavator(33t over)',
        'R16-7' : 'Mini Excavator(0.1~1.6t)',
        'R22-7' : 'Mini Excavator(1.7~2.5t)',
        'R35Z-7' : 'Mini Excavator(2.6~4t)',
        'R35Z-7A' : 'Mini Excavator(2.6~4t)',
        'R36N-7' : 'Mini Excavator(2.6~4t)',
        'R55-7' : 'Excavator(4.5~6t)',
        'R55-7A' : 'Excavator(4.5~6t)',
        'R55W-7' : 'Excavator(4.5~6t)',
        'R55W-7A' : 'Excavator(4.5~6t)',
        'R75-7' : 'Excavator(7~11t)',
        'R80-7' : 'Excavator(7~11t)',
        'R80-7A' : 'Excavator(7~11t)',
        'R15-7' : 'Mini Excavator(0.1~1.6t)',
        'R28-7' : 'Mini Excavator(2.6~4t)',
        'R35-7' : 'Mini Excavator(2.6~4t)',
        'R35Z-7' : 'Mini Excavator(2.6~4t)',
        'R555-7' : 'Excavator(4.5~6t)',
        'R555MVP' : 'Excavator(4.5~6t)',
        'R555-7K' : 'Excavator(4.5~6t)',
        'R555M' : 'Excavator(4.5~6t)',
        'R555M' : 'Excavator(4.5~6t)',
        'HX600W' : 'Excavator(4.5~6t)',
        'R1400LC-7' : 'Excavator(12~15.5t)',
        'R2200LC-7' : 'Excavator(20~27t)',
        'R2200LC-7' : 'Excavator(20~27t)',
        'R2900LC-3' : 'Excavator(28~32t)',
        'R1300LC-3TA' : 'Excavator(12~15.5t)',
        'R2200LC-3TA' : 'Excavator(20~27t)',
        'R2500LC-3' : 'Excavator(20~27t)',
        'R2900LC-3C' : 'Excavator(28~32t)',
        'R3600LC-3' : 'Excavator(33t over)',
        'R140LC' : 'Excavator(12~15.5t)',
        'R220LC' : 'Excavator(20~27t)',
        'R300LC' : 'Excavator(28~32t)',
        'R380LC' : 'Excavator(33t over)',
        'R480LC' : 'Excavator(33t over)',
        'R520LC' : 'Excavator(33t over)',
        'R8000LC-7A' : 'Excavator(33t over)',
        'R3300LC-3' : 'Excavator(33t over)',
        'R4500-7' : 'Excavator(33t over)',
        'R555-7' : 'Excavator(4.5~6t)',
        'R2200LC-7' : 'Excavator(20~27t)',
        'R1400LC-7' : 'Excavator(12~15.5t)',
        'R550-3' : 'Excavator(33t over)',
        'R555MVP' : 'Excavator(4.5~6t)',
        'R555-7' : 'Excavator(4.5~6t)',
        'R555W-7' : 'Excavator(4.5~6t)',
        'R1300WM' : 'Excavator(12~15.5t)',
        'R1300W3' : 'Excavator(12~15.5t)',
        'R2000W3' : 'Excavator(20~27t)',
        'R2500LC3' : 'Excavator(20~27t)',
        'R2900LC-7' : 'Excavator(28~32t)',
        'R3000LC-7' : 'Excavator(28~32t)',
        'R3600LC-7' : 'Excavator(33t over)',
        'R45' : 'Excavator(4.5~6t)',
        'R47' : 'Excavator(4.5~6t)',
        'R5000LC7' : 'Excavator(33t over)',
        'R1400LC-7A' : 'Excavator(12~15.5t)',
        'R1400W7-A' : 'Excavator(12~15.5t)',
        'R2200LC-7A' : 'Excavator(20~27t)',
        'R3000LC-7A' : 'Excavator(28~32t)',
        'R3700LC-7A' : 'Excavator(33t over)',
        'R4700LC-7A' : 'Excavator(33t over)',
        'HE360LCH' : 'Excavator(33t over)',
        'R140W' : 'Excavator(12~15.5t)',
        'R1700W-7A' : 'Excavator(16~19t)',
        'R2000W-7A' : 'Excavator(20~27t)',
        'R555W-7' : 'Excavator(4.5~6t)',
        'R1400W-7' : 'Excavator(12~15.5t)',
        'R1700W-7' : 'Excavator(16~19t)',
        'R2000W-7' : 'Excavator(20~27t)',
        'R110-7' : 'Excavator(7~11t)',
        'R110-7A' : 'Excavator(7~11t)',
        'R140LC-7' : 'Excavator(12~15.5t)',
        'R140LC-7A' : 'Excavator(12~15.5t)',
        'R160LC-7' : 'Excavator(16~19t)',
        'R160LC-7A' : 'Excavator(16~19t)',
        'R180LC-7' : 'Excavator(16~19t)',
        'R180LC-7A' : 'Excavator(16~19t)',
        'R210LC-7' : 'Excavator(20~27t)',
        'R210LC-7LR' : 'Excavator(20~27t)',
        'R210LC-7A' : 'Excavator(20~27t)',
        'R210LC-7ALR' : 'Excavator(20~27t)',
        'R210LC-7H' : 'Excavator(20~27t)',
        'R210NLC-7A' : 'Excavator(20~27t)',
        'R250LC-7' : 'Excavator(20~27t)',
        'R250LC-7A' : 'Excavator(20~27t)',
        'R290LC-7A' : 'Excavator(28~32t)',
        'R290LC-7ALR' : 'Excavator(28~32t)',
        'R305LC-7' : 'Excavator(28~32t)',
        'R320LC-7' : 'Excavator(28~32t)',
        'R320LC-7A' : 'Excavator(28~32t)',
        'R360LC-7A' : 'Excavator(33t over)',
        'R450LC-7' : 'Excavator(33t over)',
        'R450LC-7A' : 'Excavator(33t over)',
        'R500LC-7' : 'Excavator(33t over)',
        'R500LC-7A' : 'Excavator(33t over)',
        'R800LC-7A' : 'Excavator(33t over)',
        'R140W-7' : 'Excavator(12~15.5t)',
        'R140W-7A' : 'Excavator(12~15.5t)',
        'R170W-7' : 'Excavator(16~19t)',
        'R170W-7A' : 'Excavator(16~19t)',
        'R200W-7' : 'Excavator(20~27t)',
        'R200W-7A' : 'Excavator(20~27t)',
        'MX6W' : 'Excavator(16~19t)',
        'MINIX026' : 'Mini Excavator(2.6~4t)',
        'MINIX030' : 'Mini Excavator(2.6~4t)',
        'SE22' : 'Mini Excavator(1.7~2.5t)',
        'SE25' : 'Mini Excavator(1.7~2.5t)',
        'MX3' : 'Excavator(4.5~6t)',
        'MX3W' : 'Excavator(4.5~6t)',
        'MX3DLX' : 'Excavator(4.5~6t)',
        'MX3A' : 'Excavator(4.5~6t)',
        'MX55' : 'Excavator(4.5~6t)',
        'MX3W-2' : 'Excavator(4.5~6t)',
        'SE40W' : 'Mini Excavator(2.6~4t)',
        '60P' : 'Excavator(4.5~6t)',
        'MX6W' : 'Excavator(16~19t)',
        'MX132W' : 'Excavator(12~15.5t)',
        'MX6W-2H' : 'Excavator(16~19t)',
        'MX6LC-2' : 'Excavator(16~19t)',
        'MX6LC' : 'Excavator(16~19t)',
        'MX132LC' : 'Excavator(12~15.5t)',
        '90P' : 'Excavator(7~11t)',
        'SE90' : 'Excavator(7~11t)',
        'MX8W' : 'Excavator(16~19t)',
        'MX8W-2' : 'Excavator(16~19t)',
        'MX8-2H' : 'Excavator(16~19t)',
        'MX8' : 'Excavator(16~19t)',
        'MX8-2' : 'Excavator(16~19t)',
        'EC15B' : 'Mini Excavator(0.1~1.6t)',
        'EC55C' : 'Excavator(4.5~6t)',
        'ECR28' : 'Mini Excavator(2.6~4t)',
        'ECR38' : 'Mini Excavator(2.6~4t)',
        'ECR88' : 'Excavator(7~11t)',
        'EW60C' : 'Excavator(4.5~6t)',
        'EW145B' : 'Excavator(12~15.5t)',
        'EC140C' : 'Excavator(12~15.5t)',
        'ECR145C' : 'Excavator(12~15.5t)',
        'EC210B' : 'Excavator(20~27t)',
        'ECR235C' : 'Excavator(20~27t)',
        'EC290B' : 'Excavator(28~32t)',
        'EC360B' : 'Excavator(33t over)',
        'EC360C' : 'Excavator(33t over)',
        'EC460B' : 'Excavator(33t over)',
        'EC460C' : 'Excavator(33t over)',
        'EC700B' : 'Excavator(33t over)',
        'SE12' : 'Mini Excavator(2.6~4t)',
        'MINIX026' : 'Mini Excavator(2.6~4t)',
        'MINIX030' : 'Mini Excavator(2.6~4t)',
        'SE22' : 'Mini Excavator(1.7~2.5t)',
        'SE25' : 'Mini Excavator(1.7~2.5t)',
        'MX3' : 'Excavator(4.5~6t)',
        'MX3A' : 'Excavator(4.5~6t)',
        'MX6LC-2' : 'Excavator(12~15.5t)',
        'MX132LC/M' : 'Excavator(12~15.5t)',
        'MX8' : 'Excavator(16~19t)',
        'MX8LC' : 'Excavator(16~19t)',
        'MX8LC-2' : 'Excavator(16~19t)',
        'MX222' : 'Excavator(16~19t)',
        'MX225' : 'Excavator(16~19t)',
        'MX10' : 'Excavator(20~27t)',
        'MX10LC-2' : 'Excavator(20~27t)',
        'MX292' : 'Excavator(28~32t)',
        'MX352LC' : 'Excavator(33t over)',
        'MX452' : 'Excavator(33t over)',
        'MX3W' : 'Excavator(4.5~6t)',
        'MX3W-2' : 'Excavator(4.5~6t)',
        'SE40W' : 'Mini Excavator(2.6~4t)',
        'MX6W' : 'Excavator(16~19t)',
        'MX6W-2' : 'Excavator(16~19t)',
        'MX6W-H' : 'Excavator(16~19t)',
        'MX132W' : 'Excavator(16~19t)',
        'MX8W' : 'Excavator(16~19t)',
        'MX8W-2' : 'Excavator(16~19t)',
        'EC15C' : 'Mini Excavator(0.1~1.6t)',
        'EC17C' : 'Mini Excavator(1.7~2.5t)',
        'EC18C' : 'Mini Excavator(1.7~2.5t)',
        'EC20C' : 'Mini Excavator(1.7~2.5t)',
        'EC27C' : 'Mini Excavator(2.6~4t)',
        'EC35C' : 'Mini Excavator(2.6~4t)',
        'EC55C' : 'Excavator(4.5~6t)',
        'EC55BPro' : 'Excavator(4.5~6t)',
        'ECR28' : 'Mini Excavator(2.6~4t)',
        'ECR38' : 'Mini Excavator(2.6~4t)',
        'ECR48C' : 'Excavator(4.5~6t)',
        'ECR58P' : 'Excavator(4.5~6t)',
        'ECR88P' : 'Excavator(7~11t)',
        'EW140D' : 'Excavator(12~15.5t)',
        'EW160D' : 'Excavator(16~19t)',
        'EX180D' : 'Excavator(16~19t)',
        'EX210D' : 'Excavator(20~27t)',
        'EW230C' : 'Excavator(20~27t)',
        'EC140D' : 'Excavator(12~15.5t)',
        'ECR145D' : 'Excavator(12~15.5t)',
        'EC160D' : 'Excavator(16~19t)',
        'EC220D' : 'Excavator(20~27t)',
        'EC235C' : 'Excavator(20~27t)',
        'EC250D' : 'Excavator(20~27t)',
        'ECR305C' : 'Excavator(28~32t)',
        'EC360C' : 'Excavator(33t over)',
        'EC380D' : 'Excavator(33t over)',
        'EC460C' : 'Excavator(33t over)',
        'EC700C' : 'Excavator(33t over)',
        'EC18C' : 'Mini Excavator(1.7~2.5t)',
        'EC55C' : 'Excavator(4.5~6t)',
        'ECR28' : 'Mini Excavator(2.6~4t)',
        'ECR38' : 'Mini Excavator(2.6~4t)',
        'ECR88P' : 'Excavator(7~11t)',
        'EW60C' : 'Excavator(4.5~6t)',
        'EW145BPrime' : 'Excavator(12~15.5t)',
        'EC140C' : 'Excavator(12~15.5t)',
        'ECR145C' : 'Excavator(12~15.5t)',
        'EC210BPrime' : 'Excavator(20~27t)',
        'ECR235C' : 'Excavator(20~27t)',
        'EC290BPrime' : 'Excavator(28~32t)',
        'EC380D' : 'Excavator(33t over)',
        'EC480D' : 'Excavator(33t over)',
        'EC15' : 'Mini Excavator(0.1~1.6t)',
        'EC25' : 'Mini Excavator(1.7~2.5t)',
        'EC55' : 'Excavator(4.5~6t)',
        'MX55A' : 'Excavator(4.5~6t)',
        'EW55' : 'Excavator(4.5~6t)',
        'MX55WS' : 'Excavator(4.5~6t)',
        'MX135WS' : 'Excavator(12~15.5t)',
        'EW130' : 'Excavator(12~15.5t)',
        'EC130' : 'Excavator(12~15.5t)',
        'MX135LS' : 'Excavator(12~15.5t)',
        'EC140' : 'Excavator(12~15.5t)',
        'MX175WS' : 'Excavator(16~19t)',
        'EW170' : 'Excavator(16~19t)',
        'MX202W' : 'Excavator(20~27t)',
        'EC210' : 'Excavator(20~27t)',
        'EC240' : 'Excavator(20~27t)',
        'MX255LS' : 'Excavator(20~27t)',
        'EC290' : 'Excavator(28~32t)',
        'MX295LS' : 'Excavator(28~32t)',
        'EC360' : 'Excavator(33t over)',
        'MX365LS' : 'Excavator(33t over)',
        'EC460' : 'Excavator(33t over)',
        'MX465LS' : 'Excavator(33t over)',
        'HE50C' : 'Excavator(4.5~6t)',
        'HE50A' : 'Excavator(4.5~6t)',
        'HE50WA' : 'Excavator(4.5~6t)',
        'HE130W' : 'Excavator(12~15.5t)',
        'HE130WA' : 'Excavator(12~15.5t)',
        'HE130LC' : 'Excavator(12~15.5t)',
        'HE200W' : 'Excavator(20~27t)',
        'HE200LC' : 'Excavator(20~27t)',
        'HE280LC' : 'Excavator(28~32t)',
        'HE360LCH' : 'Excavator(33t over)',
        'HE205W' : 'Excavator(20~27t)',
        'HE220LC' : 'Excavator(20~27t)',
        'HE280' : 'Excavator(28~32t)',
        'EX120WD' : 'Excavator(12~15.5t)',
        'EX200LC' : 'Excavator(20~27t)',
        'EX220LC' : 'Excavator(20~27t)',
        'EX320LCH' : 'Excavator(28~32t)',
        'EX450LCH' : 'Excavator(33t over)',
        'HE280' : 'Excavator(28~32t)',
        'FH300' : 'Excavator(28~32t)',
        'FH400' : 'Excavator(33t over)',
        'EX45' : 'Excavator(4.5~6t)',
        'PC20MR' : 'Mini Excavator(1.7~2.5t)',
        'PC20MRX' : 'Mini Excavator(1.7~2.5t)',
        'PC30MR' : 'Mini Excavator(2.6~4t)',
        'PC30MRX' : 'Mini Excavator(2.6~4t)',
        'PC300-6Excel' : 'Excavator(28~32t)',
        'PC300LC-6Excel' : 'Excavator(28~32t)',
        'KB45' : 'Excavator(4.5~6t)',
        'DAPAR100W' : 'Excavator(7~11t)',
        'DHE-708' : 'Excavator(33t over)',
        'JACO-007' : 'Mini Excavator(0.1~1.6t)',
        'S018-VT' : 'Mini Excavator(1.7~2.5t)',
        'DX18' : 'Mini Excavator(1.7~2.5t)',
        'ECR235D' : 'Excavator(20~27t)',
        'EC180D' : 'Excavator(16~19t)',
        'EC300D' : 'Excavator(28~32t)',
        'EC480D' : 'Excavator(33t over)'
    }



    output.modelTonFilter = ["10", "08", "07", "06", "04", "03", "02", "030", "035", "017", "020", "010", "015"];
    output.modelTonName = [
        "Excavator(28~32t)",
        "Excavator(20~27t)",
        "Excavator(16~19t)",
        "Excavator(12~15.5t)",
        "Excavator(7~11t)",
        "Excavator(4.5~6t)",
        "Excavator(4.5~6t)",
        "Mini Excavator(2.6~4t)",
        "Mini Excavator(2.6~4t)",
        "Mini Excavator(1.7~2.5t)",
        "Mini Excavator(1.7~2.5t)",
        "Mini Excavator(0.1~1.6t)",
        "Mini Excavator(0.1~1.6t)"
    ];

    output.searchKeyword = {
        'Side Breaker': {
            '쁘레카': 530,
            '브레카': 380,
            '포크레인뿌레카': 30,
            '굴삭기브레카': 30,
            '브레이카': 160,
            '브레커': 100,
            '굴삭기뿌레카': 100,
            '중장비뿌레카': 10,
            '포크레인브레카': 10,
            '뿌레카': 1300,
            '굴삭기브레이카': 10,
            '굴삭기브레이커': 110,
            '포크레인브레이커': 10
        },
        'Top Breaker': {
            '쁘레카': 530,
            '브레카': 380,
            '포크레인뿌레카': 30,
            '굴삭기브레카': 30,
            '브레이카': 160,
            '브레커': 100,
            '굴삭기뿌레카': 100,
            '중장비뿌레카': 10,
            '포크레인브레카': 10,
            '뿌레카': 1300,
            '굴삭기브레이카': 10,
            '굴삭기브레이커': 110,
            '포크레인브레이커': 10
        },
        'Box Breaker': {
            '쁘레카': 530,
            '브레카': 380,
            '포크레인뿌레카': 30,
            '굴삭기브레카': 30,
            '브레이카': 160,
            '브레커': 100,
            '굴삭기뿌레카': 100,
            '중장비뿌레카': 10,
            '포크레인브레카': 10,
            '뿌레카': 1300,
            '굴삭기브레이카': 10,
            '굴삭기브레이커': 110,
            '포크레인브레이커': 10
        },
        'Special Breaker': {
            '쁘레카': 530,
            '브레카': 380,
            '포크레인뿌레카': 30,
            '굴삭기브레카': 30,
            '브레이카': 160,
            '브레커': 100,
            '굴삭기뿌레카': 100,
            '중장비뿌레카': 10,
            '포크레인브레카': 10,
            '뿌레카': 1300,
            '굴삭기브레이카': 10,
            '굴삭기브레이커': 110,
            '포크레인브레이커': 10
        },
        'Etc Breaker': {
            '쁘레카': 530,
            '브레카': 380,
            '포크레인뿌레카': 30,
            '굴삭기브레카': 30,
            '브레이카': 160,
            '브레커': 100,
            '굴삭기뿌레카': 100,
            '중장비뿌레카': 10,
            '포크레인브레카': 10,
            '뿌레카': 1300,
            '굴삭기브레이카': 10,
            '굴삭기브레이커': 110,
            '포크레인브레이커': 10
        },
        "Lift Fork": {
            '굴삭기지게발': 50,
            '지겟발': 40,
            '지게발': 330,
            '지게차덧발': 300,
            '지개발': 10,
            '지게덧발': 10,
            '중장비지게발': 10,
            '포크레인지게발': 10
        },
        "Ripper": {
            '니빠': 250,
            '니뻐': 130,
            '리빠': 90,
            '닛빠': 60,
            '리뻐': 30,
            '굴삭기니빠': 10,
            '포크레인니빠': 10,
            '중장비리퍼': 10,
            '굴삭기리퍼': 70,
            '포크레인리퍼': 10
        },

        "Hydraulic Crusher": {
            '크라샤': 920,
            '크락샤': 340,
            '크러셔': 280,
            '크랴샤': 51,
            '크랏샤': 50,
            '굴삭기크라샤': 30,
            '굴삭기크러셔': 10,
            '굴삭기크락샤': 20,
            '포크레인크라샤': 10,
            '반자동크라샤': 10
        },
        "Hydraulic Pulverizer": {
            '크라샤': 920,
            '크락샤': 340,
            '크러셔': 280,
            '크랴샤': 51,
            '크랏샤': 50,
            '굴삭기크라샤': 30,
            '굴삭기크러셔': 10,
            '굴삭기크락샤': 20,
            '포크레인크라샤': 10,
            '멍텅구리크라샤': 10
        },
        "Hydraulic Multi Processor": {
            '크라샤': 920,
            '크락샤': 340,
            '크러셔': 280,
            '크랴샤': 51,
            '크랏샤': 50,
            '굴삭기크라샤': 30,
            '굴삭기크러셔': 10,
            '굴삭기크락샤': 20,
            '포크레인크라샤': 10,
            '회전식크러셔': 10
        },
        "Etc Crusher": {
            '크라샤': 920,
            '크락샤': 340,
            '크러셔': 280,
            '크랴샤': 51,
            '크랏샤': 50,
            '굴삭기크라샤': 30,
            '굴삭기크러셔': 10,
            '굴삭기크락샤': 20,
            '포크레인크라샤': 10,
        },

        "Standard Bucket": {
            '굴삭기버켓': 530,
            '굴삭기바가지': 340,
            '포크레인바가지': 60,
            '포크레인버켓': 40,
            '중장비버켓': 10,
            '본바가지': 10,
            '본버켓': 10
        },

        "Small Bucket": {
            '굴삭기버켓': 530,
            '굴삭기바가지': 340,
            '포크레인바가지': 60,
            '포크레인버켓': 40,
            '중장비버켓': 10,
            '쪽바가지': 40,
            '쪽버켓': 10,
            '쪽박아지': 10,
            '쪽버캣': 10,
            '굴삭기쪽바가지': 10,
            '굴삭기쪽버켓': 10,
            '굴삭기망바가지': 10,
            '포크레인망바가지': 10,
            '포크레인망버켓': 10
        },
        "Skeleton Bucket": {
            '굴삭기버켓': 530,
            '굴삭기바가지': 340,
            '포크레인바가지': 60,
            '포크레인버켓': 40,
            '중장비버켓': 10,
            '채바가지': 90,
            '체바가지': 30,
            '채버켓': 30,
            '포크레인채바가지': 10,
            '굴삭기채바가지': 20,
            '중장비채바가지': 10,
            '망바가지': 20,
            '망버켓': 10,
        },
        "Wide Bucket": {
            '굴삭기버켓': 530,
            '굴삭기바가지': 340,
            '포크레인바가지': 60,
            '포크레인버켓': 40,
            '중장비버켓': 10,
            '대바가지': 30,
            '대버켓': 10,
            '굴삭기대바가지': 10,
            '굴삭기대버켓': 10,
            '포크레인대바가지': 10
        },
        "Special Bucket": {
            '포크레인바가지': 60,
            '포크레인버켓': 40,
            '중장비버켓': 10,
            '대바가지': 30,
            '대버켓': 10,
            '굴삭기대바가지': 10,
            '굴삭기대버켓': 10,
            '포크레인대바가지': 10
        },

        "Quick Coupler": {
            '크램프': 930,
            '퀵커플러': 510,
            '클렘프': 290,
            '굴삭기링크': 60,
            '크렘프': 40,
            '굴삭기퀵커플러': 30,
            '버켓링크': 20,
            '굴삭기크램프': 20,
            '버켓클램프': 10,
            '바가지링크': 10,
            '굴삭기커플러': 10,
            '포크레인링크': 10,
            '중장비링크': 10
        },

        "Orange Grapple": {
            '오렌지그라플': 10,
            '오렌지그래플': 10,
            '오렌지그랩': 10,
            '오렌지그랍': 10,
            '오랜지그랩': 10,
            '오랜지그래플': 10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "ClamShell Grapple": {
            '폐기물그랩': 10,
            '폐기물집게': 10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "Thumb Grapple": {
            '딱다구리집게': 30,
            '딱따구리집게': 20,
            '굴삭기딱따구리집게': 10,
            '굴삭기딱다구리집게': 10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "Progressive Thumb Grapple": {
            '호랑이집게': 20,
            '독수리집게': 10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "Advanced Grapple": {
            '코끼리집게': 30,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },


        "Stone Grapple": {
            '돌집게': 130,
            '스톤그랩': 10,
            '돌집개': 10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10

        },

        "Wood Grapple": {
            '우드그랩':	70,
            '굴삭기우드그랩':	10,
            '우드그래플':	10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "Etc Grapple":{
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10,
            '회전집게':	80,
            '굴삭기회전집게':	20,
            '회전집개':	10,
            '회전식집게':	10,
            '회전식집개':	10,
            '스마트집게'	:10,
            '굴삭기스마트집게'	:10,
            '포크레인회전집게':	10
        },
        "Multi Grapple":{
            '무유압집게':	60,
            '굴삭기무유압집게':	10,
            '멀티그랩':	10,
            '멀티그래플':	10,
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10
        },

        "Scrap Grapple": {
            '굴삭기집게': 140,
            '그래플': 130,
            '포크레인집게': 90,
            '그라플': 60,
            '중장비집게': 10,
            '고철집게': 30,
            '굴삭기고철집게': 10,
            '고철집개': 10,
            '포크레인고철집게': 10
        }
    }
    output.classify = {
        '쪽버켓': 'Bucket:Small Bucket',
        '본버켓': 'Bucket:Standard Bucket',
        '대버켓': 'Bucket:Wide Bucket',
        '채버켓': 'Bucket:Skeleton Bucket',
        '스페셜버켓': 'Bucket:Special Bucket',
        '기타버켓': 'Bucket:Etc Bucket',
        '버켓클렘프': 'Quick Coupler',
        '리퍼': 'Ripper',
        '무유압집게': 'Grapple:Multi Grapple',
        '고철집게': 'Grapple:Scrap Grapple',
        '돌집게': 'Grapple:Stone Grapple',
        '나무집게': 'Grapple:Wood Grapple',
        '오렌지집게': 'Grapple:Orange Grapple',
        '클렘집게': 'Grapple:ClamShell Grapple',
        '딱다구리집게': 'Grapple:Thumb Grapple',
        '호랑이집게': 'Grapple:Progressive Thumb Grapple',
        '코끼리집게': 'Grapple:Advanced Grapple',
        '기타집게': 'Grapple:Etc Grapple',
        '반자동크러셔': 'Crusher:Hydraulic Crusher',
        '멍텅구리크러셔': 'Crusher:Hydraulic Pulverizer',
        '회전식크러셔': 'Crusher:Hydraulic Multi Processor',
        '기타크러셔': 'Crusher:Etc Crusher',
        '지게발': 'Lift Fork',
        '고철가위&커터': 'Shear&Cutter',
        '해머': 'Vibro Hammer',
        '코어드릴': 'Core Drill',
        '콤펙터': 'Compactor',
        '기타': 'ETC'
    };

    output.categorys = {
        //버켓클램프
        '버켓클램프': 'Quick Coupler',
        '버캣클렘프': 'Quick Coupler',
        '버켓크렘프': 'Quick Coupler',
        '버캣크렘프': 'Quick Coupler',
        '클렘프': 'Quick Coupler',
        '크렘프': 'Quick Coupler',
        '바가지링크': 'Quick Coupler',
        '버켓링크': 'Quick Coupler',
        '클렘프버켓': 'Quick Coupler',
        '크렘프버켓': 'Quick Coupler',
        '집게바가지': 'Quick Coupler',
        '퀵커플러': 'Quick Coupler',
        '커플러': 'Quick Coupler',
        '버캣크램프': 'Quick Coupler',
        '버캣클램프': 'Quick Coupler',
        '클램프': 'Quick Coupler',
        '크램프': 'Quick Coupler',
        '버캣링크': 'Quick Coupler',
        '클램프버켓': 'Quick Coupler',
        '크램프버켓': 'Quick Coupler',
        '집개바가지': 'Quick Coupler',
        '바가지링': 'Quick Coupler',
        '버캣링': 'Quick Coupler',
        '링크': 'Quick Coupler',
        '크렌': 'Quick Couple',

        //리퍼
        '리퍼': 'Ripper',
        '리빠': 'Ripper',
        '닛빠': 'Ripper',
        '니빠': 'Ripper',
        '니퍼': 'Ripper',
        '닛바': 'Ripper',

        //지게발
        '지개발': 'Lift Fork',
        '지게발': 'Lift Fork',
        '지게차덧발': 'Lift Fork',
        '지개차덧발': 'Lift Fork',
        '지게덧발': 'Lift Fork',
        '지개덧발': 'Lift Fork',
        '덧발': 'Lift Fork',
        '지겟발': 'Lift Fork',
        '지갯발': 'Lift Fork',
        '제겟발': 'Lift Fork',
        '제갯발': 'Lift Fork',
        '재겟발': 'Lift Fork',
        '재갯발': 'Lift Fork',

        //고철가위&커터
        '가위': 'Shear&Cutter',
        '쉐어': 'Shear&Cutter',
        '빔커터': 'Shear&Cutter',
        '빔카타': 'Shear&Cutter',
        '커터기': 'Shear&Cutter',
        '카타기': 'Shear&Cutter',
        '경량가위': 'Shear&Cutter',
        '빔가위': 'Shear&Cutter',
        '커터': 'Shear&Cutter',
        '카터기': 'Shear&Cutter',
        '카터': 'Shear&Cutter',
        '캇팅기': 'Shear&Cutter',
        '컷팅기': 'Shear&Cutter',
        '캇타기': 'Shear&Cutter',

        //해머
        '해머': 'Vibro Hammer',
        '유압함마': 'Vibro Hammer',
        '유압해머': 'Vibro Hammer',
        '바이브로함마': 'Vibro Hammer',
        '바이브로해머': 'Vibro Hammer',
        '함머': 'Vibro Hammer',
        '유압함머': 'Vibro Hammer',
        '바이브로함머': 'Vibro Hammer',

        //코어드릴
        '코아드릴': 'Core Drill',
        '콘크리트절단기': 'Core Drill',
        '콘크리트절단': 'Core Drill',
        '코아': 'Core Drill',
        '코어': 'Core Drill',
        '건식코아': 'Core Drill',
        '건식코어': 'Core Drill',
        '절단기': 'Core Drill',

        //콤팩터
        '콤펙타': 'Compactor',
        '콤팩터': 'Compactor',
        '콤팩타': 'Compactor',
        '다짐': 'Compactor',
        '다짐기': 'Compactor',
        '흙다짐기': 'Compactor',
        '땅다짐기': 'Compactor',
        '콤팩트': 'Compactor',
        '콤펙트': 'Compactor',
        '콤펙': 'Compactor',
        '콤팩': 'Compactor',

        //기타
        // ETC이던애들인데 빼라고하여서 일단 공백
        '면삭기': '',
        '예초기': '',

        //==================================================================================================

        //쪽버켓
        '쪽바가지': 'Bucket/Small Bucket',
        '쪽박아지': 'Bucket/Small Bucket',
        '쪽버캣': 'Bucket/Small Bucket',
        '쪽박': 'Bucket/Small Bucket',
        '쪽바케스': 'Bucket/Small Bucket',
        '쪽버케스': 'Bucket/Small Bucket',
        '쪽바캣': 'Bucket/Small Bucket',
        '쪽바켓': 'Bucket/Small Bucket',
        '족바가지': 'Bucket/Small Bucket',
        '족박아지': 'Bucket/Small Bucket',
        '족버캣': 'Bucket/Small Bucket',
        '족박': 'Bucket/Small Bucket',
        '족바케스': 'Bucket/Small Bucket',
        '족버케스': 'Bucket/Small Bucket',
        '족바캣': 'Bucket/Small Bucket',
        '족바켓': 'Bucket/Small Bucket',

        //본버켓
        '본바가지': 'Bucket/Standard Bucket',
        '본박아지': 'Bucket/Standard Bucket',
        '본버캣': 'Bucket/Standard Bucket',
        '본박': 'Bucket/Standard Bucket',
        '본바케스': 'Bucket/Standard Bucket',
        '본버케스': 'Bucket/Standard Bucket',
        '본바캣': 'Bucket/Standard Bucket',
        '본바켓': 'Bucket/Standard Bucket',
        '일반버켓': 'Bucket/Standard Bucket',
        '일반버캣': 'Bucket/Standard Bucket',

        //대버켓
        '대바가지': 'Bucket/Wide Bucket',
        '대박아지': 'Bucket/Wide Bucket',
        '대버캣': 'Bucket/Wide Bucket',
        '대박': 'Bucket/Wide Bucket',
        '대바케스': 'Bucket/Wide Bucket',
        '대버케스': 'Bucket/Wide Bucket',
        '대바캣': 'Bucket/Wide Bucket',
        '대바켓': 'Bucket/Wide Bucket',

        //채버켓
        '채바가지': 'Bucket/Skeleton Bucket',
        '채박아지': 'Bucket/Skeleton Bucket',
        '채박': 'Bucket/Skeleton Bucket',
        '채바케스': 'Bucket/Skeleton Bucket',
        '채버케스': 'Bucket/Skeleton Bucket',
        '채바캣': 'Bucket/Skeleton Bucket',
        '채바켓': 'Bucket/Skeleton Bucket',
        '체바가지': 'Bucket/Skeleton Bucket',
        '체박아지': 'Bucket/Skeleton Bucket',
        '체버캣': 'Bucket/Skeleton Bucket',
        '체박': 'Bucket/Skeleton Bucket',
        '체바케스': 'Bucket/Skeleton Bucket',
        '체버케스': 'Bucket/Skeleton Bucket',
        '체바캣': 'Bucket/Skeleton Bucket',
        '체바켓': 'Bucket/Skeleton Bucket',
        '망버켓': 'Bucket/Skeleton Bucket',
        '망바가지': 'Bucket/Skeleton Bucket',
        '망박아지': 'Bucket/Skeleton Bucket',
        '망버캣': 'Bucket/Skeleton Bucket',
        '망박': 'Bucket/Skeleton Bucket',
        '망바케스': 'Bucket/Skeleton Bucket',
        '망버케스': 'Bucket/Skeleton Bucket',
        '망바캣': 'Bucket/Skeleton Bucket',
        '망바켓': 'Bucket/Skeleton Bucket',
        '채': 'Bucket/Skeleton Bucket',
        '망': 'Bucket/Skeleton Bucket',

        //스페셜버켓
        '스페셜바가지': 'Bucket/Special Bucket',
        '스페셜박아지': 'Bucket/Special Bucket',
        '스페셜버캣': 'Bucket/Special Bucket',
        '스페셜박': 'Bucket/Special Bucket',
        '스페셜바케스': 'Bucket/Special Bucket',
        '스페셜버케스': 'Bucket/Special Bucket',
        '스페셜바캣': 'Bucket/Special Bucket',
        '스페셜바켓': 'Bucket/Special Bucket',
        '암바가지': 'Bucket/Special Bucket',
        '암박아지': 'Bucket/Special Bucket',
        '암버캣': 'Bucket/Special Bucket',
        '암박아지': 'Bucket/Special Bucket',
        '암바케스': 'Bucket/Special Bucket',
        '암버케스': 'Bucket/Special Bucket',
        '암바캣': 'Bucket/Special Bucket',
        '암바켓': 'Bucket/Special Bucket',
        '암박': 'Bucket/Special Bucket',
        '암버켓': 'Bucket/Special Bucket',

        //기타버켓
        '기타바가지': 'Bucket/Etc Bucket',
        '기타박아지': 'Bucket/Etc Bucket',
        '기타버캣': 'Bucket/Etc Bucket',
        '기타박': 'Bucket/Etc Bucket',
        '기타바케스': 'Bucket/Etc Bucket',
        '기타버케스': 'Bucket/Etc Bucket',
        '기타바캣': 'Bucket/Etc Bucket',
        '기타바켓': 'Bucket/Etc Bucket',
        '토사': 'Bucket/Etc Bucket',
        '모래': 'Bucket/Etc Bucket',
        '손바가지': 'Bucket/Etc Bucket',
        '손버': 'Bucket/Etc Bucket',
        '암박': 'Bucket/Etc Bucket',
        '암버': 'Bucket/Etc Bucket',
        '평박': 'Bucket/Etc Bucket',
        '캐타': 'Bucket/Etc Bucket',
        '나라': 'Bucket/Etc Bucket',
        '중버': 'Bucket/Etc Bucket',
        '중바': 'Bucket/Etc Bucket',
        '중박': 'Bucket/Etc Bucket',

        //사이드뿌레카
        '사이드뿌레카': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘레카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '사이드브레이카': 'Hydraulic Breaker/Side Breaker',
        '사이드브레카': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘래카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌래카': 'Hydraulic Breaker/Side Breaker',
        '사이드브래카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '사이드부레카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌레커': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘레커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '사이드브레커': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘래커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌래커': 'Hydraulic Breaker/Side Breaker',
        '사이드브래커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '사이드부레커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌': 'Hydraulic Breaker/Side Breaker',
        '사이드형뿌': 'Hydraulic Breaker/Side Breaker',
        '사이드형브': 'Hydraulic Breaker/Side Breaker',
        '사이드부렉카': 'Hydraulic Breaker/Side Breaker',

        //탑뿌레카
        '탑뿌레카': 'Hydraulic Breaker/Top Breaker',
        '탑쁘레카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉카': 'Hydraulic Breaker/Top Breaker',
        '탑브레이카': 'Hydraulic Breaker/Top Breaker',
        '탑브레카': 'Hydraulic Breaker/Top Breaker',
        '탑쁘래카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌래카': 'Hydraulic Breaker/Top Breaker',
        '탑브래카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉카': 'Hydraulic Breaker/Top Breaker',
        '탑부레카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌레커': 'Hydraulic Breaker/Top Breaker',
        '탑쁘레커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉커': 'Hydraulic Breaker/Top Breaker',
        '탑브레커': 'Hydraulic Breaker/Top Breaker',
        '탑쁘래커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌래커': 'Hydraulic Breaker/Top Breaker',
        '탑브래커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉커': 'Hydraulic Breaker/Top Breaker',
        '탑부레커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌': 'Hydraulic Breaker/Top Breaker',
        '탑형뿌': 'Hydraulic Breaker/Top Breaker',
        '탑형브': 'Hydraulic Breaker/Top Breaker',
        '탑형': 'Hydraulic Breaker/Top Breaker',
        '탑부렉카': 'Hydraulic Breaker/Top Breaker',

        //박스뿌레카
        '박스뿌레카': 'Hydraulic Breaker/Box Breaker',
        '박스쁘레카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉카': 'Hydraulic Breaker/Box Breaker',
        '박스브레이카': 'Hydraulic Breaker/Box Breaker',
        '박스브레카': 'Hydraulic Breaker/Box Breaker',
        '박스쁘래카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌래카': 'Hydraulic Breaker/Box Breaker',
        '박스브래카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉카': 'Hydraulic Breaker/Box Breaker',
        '박스부레카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌레커': 'Hydraulic Breaker/Box Breaker',
        '박스쁘레커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉커': 'Hydraulic Breaker/Box Breaker',
        '박스브레커': 'Hydraulic Breaker/Box Breaker',
        '박스쁘래커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌래커': 'Hydraulic Breaker/Box Breaker',
        '박스브래커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉커': 'Hydraulic Breaker/Box Breaker',
        '박스부레커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌': 'Hydraulic Breaker/Box Breaker',
        '박스형뿌': 'Hydraulic Breaker/Box Breaker',
        '박스형브': 'Hydraulic Breaker/Box Breaker',
        '박스부렉카': 'Hydraulic Breaker/Box Breaker',

        //스페셜뿌레카
        '스페셜뿌레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레이카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉카': 'Hydraulic Breaker/Special Breaker',
        '스페셜부레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉커': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜브래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉커': 'Hydraulic Breaker/Special Breaker',
        '스페셜부레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌': 'Hydraulic Breaker/Special Breaker',
        '스페셜부렉카': 'Hydraulic Breaker/Special Breaker',

        //기타뿌레카
        '기타뿌레카': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘레카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉카': 'Hydraulic Breaker/Etc Breaker',
        '기타브레이카': 'Hydraulic Breaker/Etc Breaker',
        '기타브레카': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘래카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌래카': 'Hydraulic Breaker/Etc Breaker',
        '기타브래카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉카': 'Hydraulic Breaker/Etc Breaker',
        '기타부레카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌레커': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘레커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉커': 'Hydraulic Breaker/Etc Breaker',
        '기타브레커': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘래커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌래커': 'Hydraulic Breaker/Etc Breaker',
        '기타브래커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉커': 'Hydraulic Breaker/Etc Breaker',
        '기타부레커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌': 'Hydraulic Breaker/Etc Breaker',
        '기타부렉카': 'Hydraulic Breaker/Etc Breaker',

        //무유압집게
        '무유압집개': 'Grapple/Multi Grapple',
        '무유압집게': 'Grapple/Multi Grapple',
        '멀티그랩': 'Grapple/Multi Grapple',
        '멀티그라플': 'Grapple/Multi Grapple',
        '멀티그렙': 'Grapple/Multi Grapple',
        '멀티그래플': 'Grapple/Multi Grapple',
        '다용도집게': 'Grapple/Multi Grapple',
        '다용도집개': 'Grapple/Multi Grapple',
        '무유압짚게': 'Grapple/Multi Grapple',
        '무유앞집개': 'Grapple/Multi Grapple',
        '유무압집게': 'Grapple/Multi Grapple',
        '유무압집개': 'Grapple/Multi Grapple',
        '유압': 'Grapple/Multi Grapple',

        //고철집게
        '고철집개': 'Grapple/Scrap Grapple',
        '고철짚게': 'Grapple/Scrap Grapple',
        '고철짚개': 'Grapple/Scrap Grapple',
        '고철집게': 'Grapple/Scrap Grapple',

        //돌집게
        '돌집개': 'Grapple/Stone Grapple',
        '돌집게': 'Grapple/Stone Grapple',
        '돌짚게': 'Grapple/Stone Grapple',
        '돌짚개': 'Grapple/Stone Grapple',
        '회전돌집게': 'Grapple/Stone Grapple',
        '회전돌집개': 'Grapple/Stone Grapple',
        '스톤그랩': 'Grapple/Stone Grapple',
        '스톤그래플': 'Grapple/Stone Grapple',
        '스톤그라플': 'Grapple/Stone Grapple',
        '스톤그랍': 'Grapple/Stone Grapple',
        '회전돌': 'Grapple/Stone Grapple',
        '돌집': 'Grapple/Stone Grapple',
        '돌': 'Grapple/Stone Grapple',

        //나무집게
        '나무집개': 'Grapple/Wood Grapple',
        '나무집게': 'Grapple/Wood Grapple',
        '나무짚게': 'Grapple/Wood Grapple',
        '나무짚개': 'Grapple/Wood Grapple',
        '회전나무집게': 'Grapple/Wood Grapple',
        '회전나무집개': 'Grapple/Wood Grapple',
        '우드그랩': 'Grapple/Wood Grapple',
        '우드그래플': 'Grapple/Wood Grapple',
        '우드그라플': 'Grapple/Wood Grapple',
        '우드그랍': 'Grapple/Wood Grapple',
        '회전나무': 'Grapple/Wood Grapple',
        '나무': 'Grapple/Wood Grapple',

        //오렌지집게
        '오렌지': 'Grapple/Orange Grapple',
        '오랜지': 'Grapple/Orange Grapple',
        '오렌지그라플': 'Grapple/Orange Grapple',
        '오렌지그래플': 'Grapple/Orange Grapple',
        '오랜지그라플': 'Grapple/Orange Grapple',
        '오랜지그래플': 'Grapple/Orange Grapple',
        '오렌지그랩': 'Grapple/Orange Grapple',
        '오랜지그랩': 'Grapple/Orange Grapple',
        '오렌지그랍': 'Grapple/Orange Grapple',
        '오랜지그랍': 'Grapple/Orange Grapple',
        '오렌지집개': 'Grapple/Orange Grapple',
        '오렌지집게': 'Grapple/Orange Grapple',

        //클렘집게
        '클렘집개': 'Grapple/ClamShell Grapple',
        '클렘집게': 'Grapple/ClamShell Grapple',
        '클램짚게': 'Grapple/ClamShell Grapple',
        '클램집개': 'Grapple/ClamShell Grapple',
        '폐기물집게': 'Grapple/ClamShell Grapple',
        '폐기물집개': 'Grapple/ClamShell Grapple',
        '폐기물그랩': 'Grapple/ClamShell Grapple',
        '폐기물그래플': 'Grapple/ClamShell Grapple',
        '폐기물그라플': 'Grapple/ClamShell Grapple',
        '회전폐기물': 'Grapple/ClamShell Grapple',

        //딱따구리집게
        '딱다구리': 'Grapple/Thumb Grapple',
        '딱따구리': 'Grapple/Thumb Grapple',
        '딱다구리집개': 'Grapple/Thumb Grapple',
        '딱따구리집게': 'Grapple/Thumb Grapple',
        '딱따구리집개': 'Grapple/Thumb Grapple',
        '딱따': 'Grapple/Thumb Grapple',
        '딱다': 'Grapple/Thumb Grapple',

        //호랑이집게
        '호랑이': 'Grapple/Progressive Thumb Grapple',
        '호랑이집개': 'Grapple/Progressive Thumb Grapple',
        '호랑이집게': 'Grapple/Progressive Thumb Grapple',
        '독수리': 'Grapple/Progressive Thumb Grapple',
        '독수리집게': 'Grapple/Progressive Thumb Grapple',
        '독수리집개': 'Grapple/Progressive Thumb Grapple',
        '독수리발톱': 'Grapple/Progressive Thumb Grapple',
        '발톱': 'Grapple/Progressive Thumb Grapple',

        //코끼리집게
        '코끼리': 'Grapple/Advanced Grapple',
        '코끼': 'Grapple/Advanced Grapple',
        '코끼리집개': 'Grapple/Advanced Grapple',
        '코끼리집게': 'Grapple/Advanced Grapple',

        //기타집게
        '기타집개': 'Grapple/Etc Grapple',
        '기타집게': 'Grapple/Etc Grapple',
        '기타짚게': 'Grapple/Etc Grapple',
        '기타짚개': 'Grapple/Etc Grapple',
        '스마트집게': 'Grapple/Etc Grapple',
        '스마트집개': 'Grapple/Etc Grapple',
        '회전집게': 'Grapple/Etc Grapple',
        '회전집개': 'Grapple/Etc Grapple',
        '회전B게': 'Grapple/Etc Grapple',
        '회전B개': 'Grapple/Etc Grapple',
        '회전쟙': 'Grapple/Etc Grapple',
        '회전찝': 'Grapple/Etc Grapple',
        '회전찦': 'Grapple/Etc Grapple',

        //반자동크러셔
        '반자동크러셔': 'Crusher/Hydraulic Crusher',
        '반자동크라샤': 'Crusher/Hydraulic Crusher',
        '반자동크락샤': 'Crusher/Hydraulic Crusher',
        '반자동크랴샤': 'Crusher/Hydraulic Crusher',
        '반자동크랏샤': 'Crusher/Hydraulic Crusher',
        '반자동크락샤': 'Crusher/Hydraulic Crusher',
        '반자동크라셔': 'Crusher/Hydraulic Crusher',

        //멍텅구리크러셔
        '멍텅구리': 'Crusher/Hydraulic Pulverizer',
        '멍텅': 'Crusher/Hydraulic Pulverizer',
        '멍텅크러셔': 'Crusher/Hydraulic Pulverizer',
        '멍텅크라샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅크락샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅크랴샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅크랏샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅크락샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅크라셔': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크라샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크락샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크랴샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크랏샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크락샤': 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크라셔': 'Crusher/Hydraulic Pulverizer',
        '텅텅': 'Crusher/Hydraulic Pulverizer',

        //회전크러셔
        '회전크러셔': 'Crusher/Hydraulic Multi Processor',
        '회전크라샤': 'Crusher/Hydraulic Multi Processor',
        '회전크락샤': 'Crusher/Hydraulic Multi Processor',
        '회전크랴샤': 'Crusher/Hydraulic Multi Processor',
        '회전크랏샤': 'Crusher/Hydraulic Multi Processor',
        '회전크락샤': 'Crusher/Hydraulic Multi Processor',
        '회전크러셔': 'Crusher/Hydraulic Multi Processor',
        '회전식': 'Crusher/Hydraulic Multi Processor',

        //기타크러셔
       '기타크러셔': 'Crusher/Etc Crusher',
        '기타크라샤': 'Crusher/Etc Crusher',
        '기타크락샤': 'Crusher/Etc Crusher',
        '기타크랴샤': 'Crusher/Etc Crusher',
        '기타크랏샤': 'Crusher/Etc Crusher',
        '기타크락샤': 'Crusher/Etc Crusher',
        '기타크라셔': 'Crusher/Etc Crusher'
    }

    output.secondCategory = {
        '뿌레카': 'Hydraulic Breaker/Side Breaker',
        '쁘레카': 'Hydraulic Breaker/Side Breaker',
        '뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '브레이카': 'Hydraulic Breaker/Side Breaker',
        '브레카': 'Hydraulic Breaker/Side Breaker',
        '쁘래카': 'Hydraulic Breaker/Side Breaker',
        '뿌래카': 'Hydraulic Breaker/Side Breaker',
        '브래카': 'Hydraulic Breaker/Side Breaker',
        '뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '부레카': 'Hydraulic Breaker/Side Breaker',
        '뿌레커': 'Hydraulic Breaker/Side Breaker',
        '쁘레커': 'Hydraulic Breaker/Side Breaker',
        '뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '브레커': 'Hydraulic Breaker/Side Breaker',
        '쁘래커': 'Hydraulic Breaker/Side Breaker',
        '뿌래커': 'Hydraulic Breaker/Side Breaker',
        '브래커': 'Hydraulic Breaker/Side Breaker',
        '뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '부레커': 'Hydraulic Breaker/Side Breaker',
        '뿌': 'Hydraulic Breaker/Side Breaker',
        'BK': 'Hydraulic Breaker/Side Breaker',
        '부렉카': 'Hydraulic Breaker/Side Breaker',
        'bk': 'Hydraulic Breaker/Side Breaker',
        '브레': 'Hydraulic Breaker/Side Breaker',
        '브렉': 'Hydraulic Breaker/Side Breaker',
        '부레': 'Hydraulic Breaker/Side Breaker',


        '바가지': 'Bucket/Standard Bucket',
        '박아지': 'Bucket/Standard Bucket',
        '버캣': 'Bucket/Standard Bucket',
        '박': 'Bucket/Standard Bucket',
        '바케스': 'Bucket/Standard Bucket',
        '버케스': 'Bucket/Standard Bucket',
        '바캣': 'Bucket/Standard Bucket',
        '버켓': 'Bucket/Standard Bucket',
        '바켓': 'Bucket/Standard Bucket',
        '벗켓': 'Bucket/Standard Bucket',
        '벗캣': 'Bucket/Standard Bucket',


        '집개': 'Grapple/Etc Grapple',
        '짚게': 'Grapple/Etc Grapple',
        '짚개': 'Grapple/Etc Grapple',
        '그랩': 'Grapple/Etc Grapple',
        '그래플': 'Grapple/Etc Grapple',
        '그라플': 'Grapple/Etc Grapple',
        '그랍': 'Grapple/Etc Grapple',
        '집': 'Grapple/Etc Grapple',
        '찝': 'Grapple/Etc Grapple',
        '찦': 'Grapple/Etc Grapple',
        '짚': 'Grapple/Etc Grapple',
        '회전돌': 'Grapple/Etc Grapple',
        '회전나무': 'Grapple/Etc Grapple',
        '폐기물': 'Grapple/Etc Grapple',
        'B게': 'Grapple/Etc Grapple',


        '크라샤': 'Crusher/Etc Crusher',
        '크락샤': 'Crusher/Etc Crusher',
        '크랴샤': 'Crusher/Etc Crusher',
        '크랏샤': 'Crusher/Etc Crusher',
        '크락샤': 'Crusher/Etc Crusher',
        '크라셔': 'Crusher/Etc Crusher',
        '크라': 'Crusher/Etc Crusher',
        '크락': 'Crusher/Etc Crusher',
        '크러': 'Crusher/Etc Crusher',
        '크럇샤': 'Crusher/Etc Crusher',
        '크럇': 'Crusher/Etc Crusher',
        '크략': 'Crusher/Etc Crusher',

    }

    output.pageName = {
        gulsak: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_2&page=1&page=',
        attach: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_1&page=1&page=',

        dump19: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g201_111&page=1&page=',
        dump16: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g202_111&page=1&page=',
        dump10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g203_112&page=1&page=',
        churera: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g204_1&page=1&page=',
        trailler: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g205_1&page=1&page=',

        mixertruck: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g301_1&page=1&page=',

        zigyechar10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_1&page=1&page=',
        zigyechar6: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g401_1&page=1&page=',
        zigyechar5: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g402_1&page=1&page=',
        zigyecharauto: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_2&page=1&page='

    }
    //게시물에서 내용따올때
    output.idPage = {
        gulsak: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_2&wr_id=',
        attach: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_1&wr_id=',

        dump19: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g201_111&wr_id=',
        dump16: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g202_111&wr_id=',
        dump10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g203_112&wr_id=',
        churera: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g204_1&wr_id=',
        trailler: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g205_1&wr_id=',

        mixertruck: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g301_1&wr_id=',

        zigyechar10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_1&wr_id=',
        zigyechar6: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g401_1&wr_id=',
        zigyechar5: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g402_1&wr_id=',
        zigyecharauto: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_2&wr_id='
    }

    output.imgLink = {
        gulsak: '/cgis/ezVsp.html?sp=_LPshP_g105_2',
        attach: '/cgis/ezVsp.html?sp=_LPshP_g105_1',

        mixertruck: '/cgis/ezVsp.html?sp=_LPshP_g301_1',
        zigyechar6: '/cgis/ezVsp.html?sp=_LPshP_g401_1',
        zigyechar5: '/cgis/ezVsp.html?sp=_LPshP_g402_1',
        zigyechar10: '/cgis/ezVsp.html?sp=_LPshP_g400_1',

        dump10: '/cgis/ezVsp.html?sp=_LPshP_g203_112',
        dump16: '/cgis/ezVsp.html?sp=_LPshP_g202_111',
        dump19: '/cgis/ezVsp.html?sp=_LPshP_g201_111'
    }

    output.partCategoryFirst = {
        attach: 'Attachment&Parts'
    }


    if (typeof module !== 'undefined' && module.exports) {
        module.exports = output;
    }
})();
