// crawlingModule/lib/dataHelper.js
(function () {

    var output = {};

    output.modelTonFilter = ["10", "08", "07", "06", "04", "03", "02", "030", "035", "017", "020", "010", "015"];
    output.modelTonName = [
        "Excavator(28~32t)",
        "Excavator(20~27t)",
        "Excavator(16~19t)",
        "Excavator(12~15.5t)",
        "Excavator(7~11t)",
        "Excavator(4.5~6t)",
        "Excavator(4.5~6t)",
        "Mini Excavator(2.6~4t)",
        "Mini Excavator(2.6~4t)",
        "Mini Excavator(1.7~2.5t)",
        "Mini Excavator(1.7~2.5t)",
        "Mini Excavator(0.1~1.6t)",
        "Mini Excavator(0.1~1.6t)"
    ];


    output.classify = {
        '쪽버켓': 'Bucket:Small Bucket',
        '본버켓': 'Bucket:Standard Bucket',
        '대버켓': 'Bucket:Wide Bucket',
        '채버켓': 'Bucket:Skeleton Bucket',
        '스페셜버켓': 'Bucket:Special Bucket',
        '기타버켓': 'Bucket:Etc Bucket',
        '버켓클렘프': 'Quick Coupler',
        '리퍼': 'Ripper',
        '무유압집게': 'Grapple:Multi Grapple',
        '고철집게': 'Grapple:Scrap Grapple',
        '돌집게': 'Grapple:Stone Grapple',
        '나무집게': 'Grapple:Wood Grapple',
        '오렌지집게': 'Grapple:Orange Grapple',
        '클렘집게': 'Grapple:ClamShell Grapple',
        '딱다구리집게': 'Grapple:Thumb Grapple',
        '호랑이집게': 'Grapple:Progressive Thumb Grapple',
        '코끼리집게': 'Grapple:Advanced Grapple',
        '기타집게': 'Grapple:Etc Grapple',
        '반자동크러셔': 'Crusher:Hydraulic Crusher',
        '멍텅구리크러셔': 'Crusher:Hydraulic Pulverizer',
        '회전식크러셔': 'Crusher:Hydraulic Multi Processor',
        '기타크러셔': 'Crusher:Etc Crusher',
        '지게발': 'Lift Fork',
        '고철가위&커터': 'Shear&Cutter',
        '해머': 'Vibro Hammer',
        '코어드릴': 'Core Drill',
        '콤펙터': 'Compactor',
        '기타': 'ETC'
    };

    output.categorys = {
        //버켓클램프
        '버켓클램프': 'Quick Coupler',
        '버캣클렘프': 'Quick Coupler',
        '버켓크렘프': 'Quick Coupler',
        '버캣크렘프': 'Quick Coupler',
        '클렘프': 'Quick Coupler',
        '크렘프': 'Quick Coupler',
        '바가지링크': 'Quick Coupler',
        '버켓링크': 'Quick Coupler',
        '클렘프버켓': 'Quick Coupler',
        '크렘프버켓': 'Quick Coupler',
        '집게바가지': 'Quick Coupler',
        '퀵커플러': 'Quick Coupler',
        '커플러': 'Quick Coupler',
        '버캣크램프': 'Quick Coupler',
        '버캣클램프': 'Quick Coupler',
        '클램프': 'Quick Coupler',
        '크램프': 'Quick Coupler',
        '버캣링크': 'Quick Coupler',
        '클램프버켓': 'Quick Coupler',
        '크램프버켓': 'Quick Coupler',
        '집개바가지': 'Quick Coupler',
        '바가지링': 'Quick Coupler',
        '버캣링': 'Quick Coupler',
        '링크': 'Quick Coupler',
        '크렌': 'Quick Couple',

        //리퍼
        '리빠': 'Ripper',
        '닛빠': 'Ripper',
        '니빠': 'Ripper',
        '니퍼': 'Ripper',

        //지게발
        '지개발': 'Lift Fork',
        '지게발': 'Lift Fork',
        '지게차덧발': 'Lift Fork',
        '지개차덧발': 'Lift Fork',
        '지게덧발': 'Lift Fork',
        '지개덧발': 'Lift Fork',
        '덧발': 'Lift Fork',
        '지겟발': 'Lift Fork',
        '지갯발': 'Lift Fork',
        '제겟발': 'Lift Fork',
        '제갯발': 'Lift Fork',
        '재겟발': 'Lift Fork',
        '재갯발': 'Lift Fork',

        //고철가위&커터
        '가위': 'Shear&Cutter',
        '쉐어': 'Shear&Cutter',
        '빔커터': 'Shear&Cutter',
        '빔카타': 'Shear&Cutter',
        '커터기': 'Shear&Cutter',
        '카타기': 'Shear&Cutter',
        '경량가위': 'Shear&Cutter',
        '빔가위': 'Shear&Cutter',
        '커터': 'Shear&Cutter',
        '카터기': 'Shear&Cutter',
        '카터': 'Shear&Cutter',
        '캇팅기': 'Shear&Cutter',
        '컷팅기': 'Shear&Cutter',
        '캇타기': 'Shear&Cutter',

        //해머
        '해머': 'Vibro Hammer',
        '유압함마': 'Vibro Hammer',
        '유압해머': 'Vibro Hammer',
        '바이브로함마': 'Vibro Hammer',
        '바이브로해머': 'Vibro Hammer',
        '함머': 'Vibro Hammer',
        '유압함머': 'Vibro Hammer',
        '바이브로함머': 'Vibro Hammer',

        //코어드릴
        '코아드릴': 'Core Drill',
        '콘크리트절단기': 'Core Drill',
        '콘크리트절단': 'Core Drill',
        '코아': 'Core Drill',
        '코어': 'Core Drill',
        '건식코아': 'Core Drill',
        '건식코어': 'Core Drill',
        '절단기': 'Core Drill',

        //콤팩터
        '콤펙타': 'Compactor',
        '콤팩터': 'Compactor',
        '콤팩타': 'Compactor',
        '다짐': 'Compactor',
        '다짐기': 'Compactor',
        '흙다짐기': 'Compactor',
        '땅다짐기': 'Compactor',
        '콤팩트': 'Compactor',
        '콤펙트': 'Compactor',
        '콤펙': 'Compactor',
        '콤팩': 'Compactor',

        //기타
        '면삭기': 'ETC',
        '예초기': 'ETC',

        //==================================================================================================

        //쪽버켓
        '쪽바가지': 'Bucket/Small Bucket',
        '쪽박아지': 'Bucket/Small Bucket',
        '쪽버캣': 'Bucket/Small Bucket',
        '쪽박': 'Bucket/Small Bucket',
        '쪽바케스': 'Bucket/Small Bucket',
        '쪽버케스': 'Bucket/Small Bucket',
        '쪽바캣': 'Bucket/Small Bucket',
        '쪽바켓': 'Bucket/Small Bucket',
        '족바가지': 'Bucket/Small Bucket',
        '족박아지': 'Bucket/Small Bucket',
        '족버캣': 'Bucket/Small Bucket',
        '족박': 'Bucket/Small Bucket',
        '족바케스': 'Bucket/Small Bucket',
        '족버케스': 'Bucket/Small Bucket',
        '족바캣': 'Bucket/Small Bucket',
        '족바켓': 'Bucket/Small Bucket',

        //본버켓
        '본바가지': 'Bucket/Standard Bucket',
        '본박아지': 'Bucket/Standard Bucket',
        '본버캣': 'Bucket/Standard Bucket',
        '본박': 'Bucket/Standard Bucket',
        '본바케스': 'Bucket/Standard Bucket',
        '본버케스': 'Bucket/Standard Bucket',
        '본바캣': 'Bucket/Standard Bucket',
        '본바켓': 'Bucket/Standard Bucket',
        '일반버켓': 'Bucket/Standard Bucket',
        '일반버캣': 'Bucket/Standard Bucket',

        //대버켓
        '대바가지': 'Bucket/Wide Bucket',
        '대박아지': 'Bucket/Wide Bucket',
        '대버캣': 'Bucket/Wide Bucket',
        '대박': 'Bucket/Wide Bucket',
        '대바케스': 'Bucket/Wide Bucket',
        '대버케스': 'Bucket/Wide Bucket',
        '대바캣': 'Bucket/Wide Bucket',
        '대바켓': 'Bucket/Wide Bucket',

        //채버켓
        '채바가지': 'Bucket/Skeleton Bucket',
        '채박아지': 'Bucket/Skeleton Bucket',
        '채박': 'Bucket/Skeleton Bucket',
        '채바케스': 'Bucket/Skeleton Bucket',
        '채버케스': 'Bucket/Skeleton Bucket',
        '채바캣': 'Bucket/Skeleton Bucket',
        '채바켓': 'Bucket/Skeleton Bucket',
        '체바가지': 'Bucket/Skeleton Bucket',
        '체박아지': 'Bucket/Skeleton Bucket',
        '체버캣': 'Bucket/Skeleton Bucket',
        '체박': 'Bucket/Skeleton Bucket',
        '체바케스': 'Bucket/Skeleton Bucket',
        '체버케스': 'Bucket/Skeleton Bucket',
        '체바캣': 'Bucket/Skeleton Bucket',
        '체바켓': 'Bucket/Skeleton Bucket',
        '망버켓': 'Bucket/Skeleton Bucket',
        '망바가지': 'Bucket/Skeleton Bucket',
        '망박아지': 'Bucket/Skeleton Bucket',
        '망버캣': 'Bucket/Skeleton Bucket',
        '망박': 'Bucket/Skeleton Bucket',
        '망바케스': 'Bucket/Skeleton Bucket',
        '망버케스': 'Bucket/Skeleton Bucket',
        '망바캣': 'Bucket/Skeleton Bucket',
        '망바켓': 'Bucket/Skeleton Bucket',
        '채': 'Bucket/Skeleton Bucket',
        '망': 'Bucket/Skeleton Bucket',

        //스페셜버켓
        '스페셜바가지': 'Bucket/Special Bucket',
        '스페셜박아지': 'Bucket/Special Bucket',
        '스페셜버캣': 'Bucket/Special Bucket',
        '스페셜박': 'Bucket/Special Bucket',
        '스페셜바케스': 'Bucket/Special Bucket',
        '스페셜버케스': 'Bucket/Special Bucket',
        '스페셜바캣': 'Bucket/Special Bucket',
        '스페셜바켓': 'Bucket/Special Bucket',
        '암바가지': 'Bucket/Special Bucket',
        '암박아지': 'Bucket/Special Bucket',
        '암버캣': 'Bucket/Special Bucket',
        '암박아지': 'Bucket/Special Bucket',
        '암바케스': 'Bucket/Special Bucket',
        '암버케스': 'Bucket/Special Bucket',
        '암바캣': 'Bucket/Special Bucket',
        '암바켓': 'Bucket/Special Bucket',
        '암박': 'Bucket/Special Bucket',
        '암버켓': 'Bucket/Special Bucket',

        //기타버켓
        '기타바가지': 'Bucket/Etc Bucket',
        '기타박아지': 'Bucket/Etc Bucket',
        '기타버캣': 'Bucket/Etc Bucket',
        '기타박': 'Bucket/Etc Bucket',
        '기타바케스': 'Bucket/Etc Bucket',
        '기타버케스': 'Bucket/Etc Bucket',
        '기타바캣': 'Bucket/Etc Bucket',
        '기타바켓': 'Bucket/Etc Bucket',
        '토사': 'Bucket/Etc Bucket',
        '모래': 'Bucket/Etc Bucket',
        '손바가지': 'Bucket/Etc Bucket',
        '손버': 'Bucket/Etc Bucket',
        '암박': 'Bucket/Etc Bucket',
        '암버': 'Bucket/Etc Bucket',
        '평박': 'Bucket/Etc Bucket',
        '캐타': 'Bucket/Etc Bucket',
        '나라': 'Bucket/Etc Bucket',
        '중버': 'Bucket/Etc Bucket',
        '중바': 'Bucket/Etc Bucket',
        '중박': 'Bucket/Etc Bucket',

        //사이드뿌레카
        '사이드뿌레카': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘레카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '사이드브레이카': 'Hydraulic Breaker/Side Breaker',
        '사이드브레카': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘래카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌래카': 'Hydraulic Breaker/Side Breaker',
        '사이드브래카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉카': 'Hydraulic Breaker/Side Breaker',
        '사이드부레카': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌레커': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘레커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '사이드브레커': 'Hydraulic Breaker/Side Breaker',
        '사이드쁘래커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌래커': 'Hydraulic Breaker/Side Breaker',
        '사이드브래커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌렉커': 'Hydraulic Breaker/Side Breaker',
        '사이드부레커': 'Hydraulic Breaker/Side Breaker',
        '사이드뿌': 'Hydraulic Breaker/Side Breaker',
        '사이드형뿌': 'Hydraulic Breaker/Side Breaker',
        '사이드형브': 'Hydraulic Breaker/Side Breaker',
        '사이드부렉카': 'Hydraulic Breaker/Side Breaker',

        //탑뿌레카
        '탑뿌레카': 'Hydraulic Breaker/Top Breaker',
        '탑쁘레카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉카': 'Hydraulic Breaker/Top Breaker',
        '탑브레이카': 'Hydraulic Breaker/Top Breaker',
        '탑브레카': 'Hydraulic Breaker/Top Breaker',
        '탑쁘래카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌래카': 'Hydraulic Breaker/Top Breaker',
        '탑브래카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉카': 'Hydraulic Breaker/Top Breaker',
        '탑부레카': 'Hydraulic Breaker/Top Breaker',
        '탑뿌레커': 'Hydraulic Breaker/Top Breaker',
        '탑쁘레커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉커': 'Hydraulic Breaker/Top Breaker',
        '탑브레커': 'Hydraulic Breaker/Top Breaker',
        '탑쁘래커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌래커': 'Hydraulic Breaker/Top Breaker',
        '탑브래커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌렉커': 'Hydraulic Breaker/Top Breaker',
        '탑부레커': 'Hydraulic Breaker/Top Breaker',
        '탑뿌': 'Hydraulic Breaker/Top Breaker',
        '탑형뿌': 'Hydraulic Breaker/Top Breaker',
        '탑형브': 'Hydraulic Breaker/Top Breaker',
        '탑형': 'Hydraulic Breaker/Top Breaker',
        '탑부렉카': 'Hydraulic Breaker/Top Breaker',

        //박스뿌레카
        '박스뿌레카': 'Hydraulic Breaker/Box Breaker',
        '박스쁘레카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉카': 'Hydraulic Breaker/Box Breaker',
        '박스브레이카': 'Hydraulic Breaker/Box Breaker',
        '박스브레카': 'Hydraulic Breaker/Box Breaker',
        '박스쁘래카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌래카': 'Hydraulic Breaker/Box Breaker',
        '박스브래카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉카': 'Hydraulic Breaker/Box Breaker',
        '박스부레카': 'Hydraulic Breaker/Box Breaker',
        '박스뿌레커': 'Hydraulic Breaker/Box Breaker',
        '박스쁘레커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉커': 'Hydraulic Breaker/Box Breaker',
        '박스브레커': 'Hydraulic Breaker/Box Breaker',
        '박스쁘래커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌래커': 'Hydraulic Breaker/Box Breaker',
        '박스브래커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌렉커': 'Hydraulic Breaker/Box Breaker',
        '박스부레커': 'Hydraulic Breaker/Box Breaker',
        '박스뿌': 'Hydraulic Breaker/Box Breaker',
        '박스형뿌': 'Hydraulic Breaker/Box Breaker',
        '박스형브': 'Hydraulic Breaker/Box Breaker',
        '박스부렉카': 'Hydraulic Breaker/Box Breaker',

        //스페셜뿌레카
        '스페셜뿌레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레이카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜브래카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉카': 'Hydraulic Breaker/Special Breaker',
        '스페셜부레카': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉커': 'Hydraulic Breaker/Special Breaker',
        '스페셜브레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜쁘래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜브래커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌렉커': 'Hydraulic Breaker/Special Breaker',
        '스페셜부레커': 'Hydraulic Breaker/Special Breaker',
        '스페셜뿌': 'Hydraulic Breaker/Special Breaker',
        '스페셜부렉카': 'Hydraulic Breaker/Special Breaker',

        //기타뿌레카
        '기타뿌레카': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘레카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉카': 'Hydraulic Breaker/Etc Breaker',
        '기타브레이카': 'Hydraulic Breaker/Etc Breaker',
        '기타브레카': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘래카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌래카': 'Hydraulic Breaker/Etc Breaker',
        '기타브래카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉카': 'Hydraulic Breaker/Etc Breaker',
        '기타부레카': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌레커': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘레커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉커': 'Hydraulic Breaker/Etc Breaker',
        '기타브레커': 'Hydraulic Breaker/Etc Breaker',
        '기타쁘래커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌래커': 'Hydraulic Breaker/Etc Breaker',
        '기타브래커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌렉커': 'Hydraulic Breaker/Etc Breaker',
        '기타부레커': 'Hydraulic Breaker/Etc Breaker',
        '기타뿌': 'Hydraulic Breaker/Etc Breaker',
        '기타부렉카': 'Hydraulic Breaker/Etc Breaker',

        //무유압집게
        '무유압집개' : 'Grapple/Multi Grapple',
        '멀티그랩' : 'Grapple/Multi Grapple',
        '멀티그라플' : 'Grapple/Multi Grapple',
        '멀티그렙' : 'Grapple/Multi Grapple',
        '멀티그래플' : 'Grapple/Multi Grapple',
        '다용도집게' : 'Grapple/Multi Grapple',
        '다용도집개' : 'Grapple/Multi Grapple',
        '무유압짚게' : 'Grapple/Multi Grapple',
        '무유앞집개' : 'Grapple/Multi Grapple',
        '유무압집게' : 'Grapple/Multi Grapple',
        '유무압집개' : 'Grapple/Multi Grapple',
        '유압' : 'Grapple/Multi Grapple',

        //고철집게
        '고철집개' : 'Grapple/Scrap Grapple',
        '고철짚게' : 'Grapple/Scrap Grapple',
        '고철짚개' : 'Grapple/Scrap Grapple',

        //돌집게
        '돌집개' : 'Grapple/Stone Grapple',
        '돌짚게' : 'Grapple/Stone Grapple',
        '돌짚개' : 'Grapple/Stone Grapple',
        '회전돌집게' : 'Grapple/Stone Grapple',
        '회전돌집개' : 'Grapple/Stone Grapple',
        '스톤그랩' : 'Grapple/Stone Grapple',
        '스톤그래플' : 'Grapple/Stone Grapple',
        '스톤그라플' : 'Grapple/Stone Grapple',
        '스톤그랍' : 'Grapple/Stone Grapple',
        '회전돌' : 'Grapple/Stone Grapple',
        '돌집' : 'Grapple/Stone Grapple',
        '돌' : 'Grapple/Stone Grapple',

        //나무집게
        '나무집개' : 'Grapple/Wood Grapple',
        '나무짚게' : 'Grapple/Wood Grapple',
        '나무짚개' : 'Grapple/Wood Grapple',
        '회전나무집게' : 'Grapple/Wood Grapple',
        '회전나무집개' : 'Grapple/Wood Grapple',
        '우드그랩' : 'Grapple/Wood Grapple',
        '우드그래플' : 'Grapple/Wood Grapple',
        '우드그라플' : 'Grapple/Wood Grapple',
        '우드그랍' : 'Grapple/Wood Grapple',
        '회전나무' : 'Grapple/Wood Grapple',
        '나무' : 'Grapple/Wood Grapple',

        //오렌지집게
        '오렌지' : 'Grapple/Orange Grapple',
        '오랜지' : 'Grapple/Orange Grapple',
        '오렌지그라플' : 'Grapple/Orange Grapple',
        '오렌지그래플' : 'Grapple/Orange Grapple',
        '오랜지그라플' : 'Grapple/Orange Grapple',
        '오랜지그래플' : 'Grapple/Orange Grapple',
        '오렌지그랩' : 'Grapple/Orange Grapple',
        '오랜지그랩' : 'Grapple/Orange Grapple',
        '오렌지그랍' : 'Grapple/Orange Grapple',
        '오랜지그랍' : 'Grapple/Orange Grapple',
        '오렌지집개' : 'Grapple/Orange Grapple',

        //클렘집게
        '클렘집개' : 'Grapple/ClamShell Grapple',
        '클램짚게' : 'Grapple/ClamShell Grapple',
        '클램집개' : 'Grapple/ClamShell Grapple',
        '폐기물집게' : 'Grapple/ClamShell Grapple',
        '폐기물집개' : 'Grapple/ClamShell Grapple',
        '폐기물그랩' : 'Grapple/ClamShell Grapple',
        '폐기물그래플' : 'Grapple/ClamShell Grapple',
        '폐기물그라플' : 'Grapple/ClamShell Grapple',
        '회전폐기물' : 'Grapple/ClamShell Grapple',

        //딱따구리집게
        '딱다구리' : 'Grapple/Thumb Grapple',
        '딱따구리' : 'Grapple/Thumb Grapple',
        '딱다구리집개' : 'Grapple/Thumb Grapple',
        '딱따구리집게' : 'Grapple/Thumb Grapple',
        '딱따구리집개' : 'Grapple/Thumb Grapple',
        '딱따' : 'Grapple/Thumb Grapple',
        '딱다' : 'Grapple/Thumb Grapple',

        //호랑이집게
        '호랑이' : 'Grapple/Progressive Thumb Grapple',
        '호랑이집개' : 'Grapple/Progressive Thumb Grapple',
        '독수리' : 'Grapple/Progressive Thumb Grapple',
        '독수리집게' : 'Grapple/Progressive Thumb Grapple',
        '독수리집개' : 'Grapple/Progressive Thumb Grapple',
        '독수리발톱' : 'Grapple/Progressive Thumb Grapple',
        '발톱' : 'Grapple/Progressive Thumb Grapple',

        //코끼리집게
        '코끼리' : 'Grapple/Advanced Grapple',
        '코끼' : 'Grapple/Advanced Grapple',
        '코끼리집개' : 'Grapple/Advanced Grapple',

        //기타집게
        '기타집개' : 'Grapple/Etc Grapple',
        '기타짚게' : 'Grapple/Etc Grapple',
        '기타짚개' : 'Grapple/Etc Grapple',
        '스마트집게' : 'Grapple/Etc Grapple',
        '스마트집개' : 'Grapple/Etc Grapple',
        '회전집게' : 'Grapple/Etc Grapple',
        '회전집개' : 'Grapple/Etc Grapple',
        '회전B게' : 'Grapple/Etc Grapple',
        '회전B개' : 'Grapple/Etc Grapple',
        '회전쟙' : 'Grapple/Etc Grapple',
        '회전찝' : 'Grapple/Etc Grapple',
        '회전찦' : 'Grapple/Etc Grapple',

        //반자동크러셔
        '반자동크라샤' : 'Crusher/Hydraulic Crusher',
        '반자동크락샤' : 'Crusher/Hydraulic Crusher',
        '반자동크랴샤' : 'Crusher/Hydraulic Crusher',
        '반자동크랏샤' : 'Crusher/Hydraulic Crusher',
        '반자동크락샤' : 'Crusher/Hydraulic Crusher',
        '반자동크라셔' : 'Crusher/Hydraulic Crusher',

        //멍텅구리크러셔
        '멍텅구리' : 'Crusher/Hydraulic Pulverizer',
        '멍텅' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크러셔' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크라샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크락샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크랴샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크랏샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크락샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅크라셔' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크라샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크락샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크랴샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크랏샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크락샤' : 'Crusher/Hydraulic Pulverizer',
        '멍텅구리크라셔' : 'Crusher/Hydraulic Pulverizer',
        '텅텅' : 'Crusher/Hydraulic Pulverizer',

        //회전크러셔
        '회전크라샤' : 'Crusher/Hydraulic Multi Processor',
        '회전크락샤' : 'Crusher/Hydraulic Multi Processor',
        '회전크랴샤' : 'Crusher/Hydraulic Multi Processor',
        '회전크랏샤' : 'Crusher/Hydraulic Multi Processor',
        '회전크락샤' : 'Crusher/Hydraulic Multi Processor',
        '회전크러셔' : 'Crusher/Hydraulic Multi Processor',
        '회전식' : 'Crusher/Hydraulic Multi Processor',

        //기타크러셔
        '기타크라샤' : 'Crusher/Etc Crusher',
        '기타크락샤' : 'Crusher/Etc Crusher',
        '기타크랴샤' : 'Crusher/Etc Crusher',
        '기타크랏샤' : 'Crusher/Etc Crusher',
        '기타크락샤' : 'Crusher/Etc Crusher',
        '기타크라셔' : 'Crusher/Etc Crusher'
    }

    output.secondCategory = {
        '뿌레카' : 'Hydraulic Breaker/Side Breaker',
        '쁘레카' : 'Hydraulic Breaker/Side Breaker',
        '뿌렉카' : 'Hydraulic Breaker/Side Breaker',
        '브레이카' : 'Hydraulic Breaker/Side Breaker',
        '브레카' : 'Hydraulic Breaker/Side Breaker',
        '쁘래카' : 'Hydraulic Breaker/Side Breaker',
        '뿌래카' : 'Hydraulic Breaker/Side Breaker',
        '브래카' : 'Hydraulic Breaker/Side Breaker',
        '뿌렉카' : 'Hydraulic Breaker/Side Breaker',
        '부레카' : 'Hydraulic Breaker/Side Breaker',
        '뿌레커' : 'Hydraulic Breaker/Side Breaker',
        '쁘레커' : 'Hydraulic Breaker/Side Breaker',
        '뿌렉커' : 'Hydraulic Breaker/Side Breaker',
        '브레커' : 'Hydraulic Breaker/Side Breaker',
        '쁘래커' : 'Hydraulic Breaker/Side Breaker',
        '뿌래커' : 'Hydraulic Breaker/Side Breaker',
        '브래커' : 'Hydraulic Breaker/Side Breaker',
        '뿌렉커' : 'Hydraulic Breaker/Side Breaker',
        '부레커' : 'Hydraulic Breaker/Side Breaker',
        '뿌' : 'Hydraulic Breaker/Side Breaker',
        'BK' : 'Hydraulic Breaker/Side Breaker',
        '부렉카' : 'Hydraulic Breaker/Side Breaker',
        'bk' : 'Hydraulic Breaker/Side Breaker',
        '브레' : 'Hydraulic Breaker/Side Breaker',
        '브렉' : 'Hydraulic Breaker/Side Breaker',
        '부레' : 'Hydraulic Breaker/Side Breaker',


        '바가지' : 'Bucket/Standard Bucket',
        '박아지' : 'Bucket/Standard Bucket',
        '버캣' : 'Bucket/Standard Bucket',
        '박' : 'Bucket/Standard Bucket',
        '바케스' : 'Bucket/Standard Bucket',
        '버케스' : 'Bucket/Standard Bucket',
        '바캣' : 'Bucket/Standard Bucket',
        '바켓' : 'Bucket/Standard Bucket',
        '벗켓' : 'Bucket/Standard Bucket',
        '벗캣' : 'Bucket/Standard Bucket',


        '집개' : 'Grapple/Etc Grapple',
        '짚게' : 'Grapple/Etc Grapple',
        '짚개' : 'Grapple/Etc Grapple',
        '그랩' : 'Grapple/Etc Grapple',
        '그래플' : 'Grapple/Etc Grapple',
        '그라플' : 'Grapple/Etc Grapple',
        '그랍' : 'Grapple/Etc Grapple',
        '집' : 'Grapple/Etc Grapple',
        '찝' : 'Grapple/Etc Grapple',
        '찦' : 'Grapple/Etc Grapple',
        '짚' : 'Grapple/Etc Grapple',
        '회전돌' : 'Grapple/Etc Grapple',
        '회전나무' : 'Grapple/Etc Grapple',
        '폐기물' : 'Grapple/Etc Grapple',
        'B게' : 'Grapple/Etc Grapple',


        '크라샤' : 'Crusher/Etc Crusher',
        '크락샤' : 'Crusher/Etc Crusher',
        '크랴샤' : 'Crusher/Etc Crusher',
        '크랏샤' : 'Crusher/Etc Crusher',
        '크락샤' : 'Crusher/Etc Crusher',
        '크라셔' : 'Crusher/Etc Crusher',
        '크라' : 'Crusher/Etc Crusher',
        '크락' : 'Crusher/Etc Crusher',
        '크러' : 'Crusher/Etc Crusher',
        '크럇샤' : 'Crusher/Etc Crusher',
        '크럇' : 'Crusher/Etc Crusher',
        '크략' : 'Crusher/Etc Crusher',

    }

    // output.secondCategory = {
    //     '브레이커': [
    //         '뿌레카', '부레', '쁘레카', '뿌렉카', '브레이카', '브레카', '쁘래카', '뿌래카', '브래카',
    //         '뿌렉카', '부레카', '뿌레커', '쁘레커', '뿌렉커', '브레커', '쁘래커', '뿌래커', '브래커',
    //         '뿌렉커', '부레커', '뿌', 'BK', '부렉카', 'bk', '브레', '브렉'
    //     ],
    //     '버켓클렘프': [
    //         '버켓클램프', '버캣클렘프', '버켓크렘프', '버캣크렘프', '클렘프', '크렘프', '바가지링크', '버켓링크',
    //         '클렘프버켓', '크렘프버켓', '집게바가지', '퀵커플러', '커플러', '버캣크램프', '버캣클램프', '클램프',
    //         '크램프', '버캣링크', '클램프버켓', '크램프버켓', '집개바가지', '바가지링', '버캣링', '링크', '크렌'
    //     ],
    //     '리퍼': [
    //         '리빠', '닛빠', '니빠', '니퍼'
    //     ],
    //     '지게발': [
    //         '지개발', '지게차덧발', '지개차덧발', '지게덧발', '지개덧발', '덧발',
    //         '지겟발', '지갯발', '제겟발', '제갯발', '재겟발', '재갯발'
    //     ],
    //     '고철가위&커터': [
    //         '가위', '쉐어', '빔커터', '빔카타', '캇팅기', '컷팅기', '캇타기',
    //         '커터기', '카타기', '경량가위', '빔가위', '커터', '카터기', '카터',
    //     ],
    //     '함마': [
    //         '해머', '유압함마', '유압해머', '바이브로함마', '바이브로해머', '함머', '유압함머', '바이브로함머'
    //     ],
    //     '코어드릴': [
    //         '코아드릴', '콘크리트절단기', '콘크리트절단', '코아',
    //         '코어', '건식코아', '건식코어', '절단기'
    //     ],
    //     '콤펙터': [
    //         '콤펙타', '콤팩타', '다짐', '다짐기', '흙다짐기',
    //         '콤팩터', '땅다짐기', '콤팩트', '콤펙트', '콤펙', '콤팩'
    //     ],
    //     '기타': ['면삭기', '예초기'],
    //     '버켓': [
    //         '바가지', '박아지', '버캣', '박', '바케스', '버케스', '바캣', '바켓', '벗켓', '벗캣'
    //     ],
    //     '집게': [
    //         '집개', '짚게', '짚개', '그랩', '그래플', '그라플', '그랍', '딱다구리', '딱따구리', '딱다', '딱따',
    //         '코끼리', '코끼', '호랑이', '독수리', '발톱', '오렌지', '오랜지', '집', '짚',
    //         '회전돌', '회전나무', '돌', '나무', '폐기물', 'B게', '찦게', '찝게'
    //     ],
    //     '크러셔': [
    //         '크라샤', '크러', '멍텅구리', '멍텅', '텅텅', '크럇샤', '크럇', '크략',
    //         '크락샤', '크랴샤', '크랏샤', '크락샤', '크라셔', '크라', '크락'
    //     ]
    // }
    //
    // output.thirdCategory = {
    //     '쪽버켓': [],
    //     '본버켓': [],
    //     '대버켓': [],
    //     '채버켓': [],
    //     '스페셜버켓': [],
    //     '기타버켓': [],
    //     '사이드브레이커': [],
    //     '탑브레이커': [],
    //     '박스브레이커': [],
    //     '스페셜브레이커': [],
    //     '기타브레이커': [],
    //     '무유압집게': [],
    //     '고철집게': [],
    //     '돌집게': [],
    //     '나무집게': [],
    //     '오렌지집게': [],
    //     '클렘집게': [],
    //     '딱다구리집게': [],
    //     '호랑이집게': [],
    //     '코끼리집게': [],
    //     '기타집게': [],
    //     '반자동크러셔': [],
    //     '멍텅구리크러셔': [],
    //     '회전식크러셔': [],
    //     '기타 크러셔': []
    // }


    output.pageName = {
        gulsak: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_2&page=1&page=',
        attach: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_1&page=1&page=',

        dump19: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g201_111&page=1&page=',
        dump16: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g202_111&page=1&page=',
        dump10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g203_112&page=1&page=',
        churera: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g204_1&page=1&page=',
        trailler: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g205_1&page=1&page=',

        mixertruck: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g301_1&page=1&page=',

        zigyechar10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_1&page=1&page=',
        zigyechar6: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g401_1&page=1&page=',
        zigyechar5: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g402_1&page=1&page=',
        zigyecharauto: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_2&page=1&page='

    }
    //게시물에서 내용따올때
    output.idPage = {
        gulsak: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_2&wr_id=',
        attach: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g105_1&wr_id=',

        dump19: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g201_111&wr_id=',
        dump16: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g202_111&wr_id=',
        dump10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g203_112&wr_id=',
        churera: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g204_1&wr_id=',
        trailler: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g205_1&wr_id=',

        mixertruck: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g301_1&wr_id=',

        zigyechar10: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_1&wr_id=',
        zigyechar6: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g401_1&wr_id=',
        zigyechar5: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g402_1&wr_id=',
        zigyecharauto: 'http://wgreen.cafe24.com/bbs/gn4/bbs/board.php?bo_table=g400_2&wr_id='
    }

    output.imgLink = {
        gulsak: '/cgis/ezVsp.html?sp=_LPshP_g105_2',
        attach: '/cgis/ezVsp.html?sp=_LPshP_g105_1',

        mixertruck: '/cgis/ezVsp.html?sp=_LPshP_g301_1',
        zigyechar6: '/cgis/ezVsp.html?sp=_LPshP_g401_1',
        zigyechar5: '/cgis/ezVsp.html?sp=_LPshP_g402_1',
        zigyechar10: '/cgis/ezVsp.html?sp=_LPshP_g400_1',

        dump10: '/cgis/ezVsp.html?sp=_LPshP_g203_112',
        dump16: '/cgis/ezVsp.html?sp=_LPshP_g202_111',
        dump19: '/cgis/ezVsp.html?sp=_LPshP_g201_111'
    }

    output.partCategoryFirst = {
        attach: 'Attachment&Parts'
    }


    if (typeof module !== 'undefined' && module.exports) {
        module.exports = output;
    }
})();
